---
annotations:
- type: Pathway Ontology
  value: '"pathway pertinent to DNA replication and repair'
authors:
- ReactomeTeam
- Mkutmon
description: 'Methylation of cytosine is catalyzed by a family of DNA methyltransferases
  (DNMTs): DNMT1, DNMT3A, and DNMT3B transfer methyl groups from S-adenosylmethionine
  to cytosine, producing 5-methylcytosine and homocysteine (reviewed in Klose and
  Bird 2006, Ooi et al. 2009, Jurkowska et al. 2011, Moore et al. 2013). (DNMT2 appears
  to methylate RNA rather than DNA.) DNMT1, the first enzyme discovered, preferentially
  methylates hemimethylated CG motifs that are produced by replication (template strand
  methylated, synthesized strand unmethylated). Thus it maintains existing methylation
  through cell division. DNMT3A and DNMT3B catalyze de novo methylation at unmethylated
  sites that include both CG dinucleotides and non-CG motifs.<br>DNA from adult humans
  contains about 0.76 to 1.00 mole percent 5-methylcytosine (Ehrlich et al. 1982,
  reviewed in Klose and Bird 2006, Ooi et al. 2009, Moore et al. 2013). Methylation
  of DNA occurs at cytosines that are mainly located in CG dinucleotides. CG dinucleotides
  are unevenly distributed in the genome. Promoter regions tend to have a high CG-content,
  forming so-called CG-islands (CGIs), while the CG-content in the remaining part
  of the genome is much lower. CGIs tend to be unmethylated, while the majority of
  CGs outside CGIs are methylated. Methylation in promoters and first exons tends
  to repress transcription while methylation in gene bodies (regions of genes downstream
  of the promoter and first exon) correlates with transcription (reviewed in Ehrlich
  and Lacey 2013, Kulis et al. 2013). Proteins such as MeCP2 and MBDs specifically
  bind 5-methylcytosine and may recruit other factors.<br>Mammalian development has
  two major episodes of genome-wide demethylation and remethylation (reviewed in Zhou
  2012, Guibert and Weber 2013, Hackett and Surani 2013, Dean 2014). In mice about
  1 day after fertilization the paternal genome is actively demethylated by TET proteins
  together with thymine DNA glycosylase and the maternal genome is demethylated by
  passive dilution during replication, however methylation at imprinted sites is maintained.
  The genome has its lowest methylation level about 3.5 days post-fertilization. Remethylation
  occurs by 6.5 days post-fertilization. The second demethylation-remethylation event
  occurs in primordial germ cells of the developing embryo about 12.5 days post-fertilization.
  DNMT3A and DNMT3B, together with the non-catalytic DNMT3L, play major roles in the
  remethylation events (reviewed in Chen and Chan 2014). How the methyltransferases
  are directed to particular regions of the genome remains an area of active research.
  The mechanisms at each locus may differ in detail but a connection between histone
  modifications and DNA methylation has been observed (reviewed in Rose and Klose
  2014).  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=5334118
  Reactome].'
last-edited: 2021-01-25
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP3359
- /instance/WP3359
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP3359.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: 'Methylation of cytosine is catalyzed by a family of DNA methyltransferases
    (DNMTs): DNMT1, DNMT3A, and DNMT3B transfer methyl groups from S-adenosylmethionine
    to cytosine, producing 5-methylcytosine and homocysteine (reviewed in Klose and
    Bird 2006, Ooi et al. 2009, Jurkowska et al. 2011, Moore et al. 2013). (DNMT2
    appears to methylate RNA rather than DNA.) DNMT1, the first enzyme discovered,
    preferentially methylates hemimethylated CG motifs that are produced by replication
    (template strand methylated, synthesized strand unmethylated). Thus it maintains
    existing methylation through cell division. DNMT3A and DNMT3B catalyze de novo
    methylation at unmethylated sites that include both CG dinucleotides and non-CG
    motifs.<br>DNA from adult humans contains about 0.76 to 1.00 mole percent 5-methylcytosine
    (Ehrlich et al. 1982, reviewed in Klose and Bird 2006, Ooi et al. 2009, Moore
    et al. 2013). Methylation of DNA occurs at cytosines that are mainly located in
    CG dinucleotides. CG dinucleotides are unevenly distributed in the genome. Promoter
    regions tend to have a high CG-content, forming so-called CG-islands (CGIs), while
    the CG-content in the remaining part of the genome is much lower. CGIs tend to
    be unmethylated, while the majority of CGs outside CGIs are methylated. Methylation
    in promoters and first exons tends to repress transcription while methylation
    in gene bodies (regions of genes downstream of the promoter and first exon) correlates
    with transcription (reviewed in Ehrlich and Lacey 2013, Kulis et al. 2013). Proteins
    such as MeCP2 and MBDs specifically bind 5-methylcytosine and may recruit other
    factors.<br>Mammalian development has two major episodes of genome-wide demethylation
    and remethylation (reviewed in Zhou 2012, Guibert and Weber 2013, Hackett and
    Surani 2013, Dean 2014). In mice about 1 day after fertilization the paternal
    genome is actively demethylated by TET proteins together with thymine DNA glycosylase
    and the maternal genome is demethylated by passive dilution during replication,
    however methylation at imprinted sites is maintained. The genome has its lowest
    methylation level about 3.5 days post-fertilization. Remethylation occurs by 6.5
    days post-fertilization. The second demethylation-remethylation event occurs in
    primordial germ cells of the developing embryo about 12.5 days post-fertilization.
    DNMT3A and DNMT3B, together with the non-catalytic DNMT3L, play major roles in
    the remethylation events (reviewed in Chen and Chan 2014). How the methyltransferases
    are directed to particular regions of the genome remains an area of active research.
    The mechanisms at each locus may differ in detail but a connection between histone
    modifications and DNA methylation has been observed (reviewed in Rose and Klose
    2014).  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=5334118
    Reactome].'
  keywords:
  - DNMT3A:DNMT3L:Chromatin
  - hemimethylC
  - DNMT1
  - 'HIST2H2BE '
  - Chromatin with
  - 'DNA '
  - 'HIST1H2BB '
  - 'DNA containing 5-mC '
  - 'hemimethylated DNA '
  - UHRF1
  - DNMT3B:DNMT3L:Chromatin
  - 'HIST1H2BJ '
  - AdoHcy
  - 'HIST2H3A '
  - 'HIST1H2BA '
  - 'H2AFZ '
  - 'H2AFB1 '
  - 'HIST1H2BN '
  - 'HIST1H2BD '
  - 'DNMT3A '
  - DNMT3B:DNMT3L:Chromatin with 5mC
  - AdoMet
  - 'H2AFX '
  - DNMT3A:DNMT3L:Chromatin with 5mC
  - DNMT3B:DNMT3L
  - 'HIST1H2AJ '
  - 'HIST1H3A '
  - 'HIST1H2BH '
  - UHRF1:Chromatin with
  - 'HIST1H2BL '
  - 'H2AFV '
  - 'HIST1H2BO '
  - 'HIST1H4 '
  - 'HIST3H2BB '
  - DNMT1:UHRF1:Chromatin with symmetric methylC
  - 'H2BFS '
  - 'HIST1H2AC '
  - DNMT1:UHRF1:Chromatin with hemimethylC
  - 'HIST1H2BK '
  - 'DNMT1 '
  - 'HIST2H2AA3 '
  - 'HIST1H2AB '
  - 'HIST1H2AD '
  - 'H2AFJ '
  - 'UHRF1 '
  - DNMT3A:DNMT3L
  - 'HIST1H2BM '
  - Chromatin
  - 'HIST2H2AC '
  - 'DNMT3B '
  - 'symmetrically methylated DNA '
  - 'DNMT3L '
  - 'H3F3A '
  - 'HIST1H2BC '
  license: CC0
  name: DNA methylation
seo: CreativeWork
title: DNA methylation
wpid: WP3359
---