---
authors:
- ReactomeTeam
description: PTEN is regulated at the level of gene transcription, mRNA translation,
  localization and protein stability.<p>Transcription of the PTEN gene is regulated
  at multiple levels. Epigenetic repression involves the recruitment of Mi-2/NuRD
  upon SALL4 binding to the PTEN promoter (Yang et al. 2008, Lu et al. 2009) or EVI1-mediated
  recruitment of the polycomb repressor complex (PRC) to the PTEN promoter (Song et
  al. 2009, Yoshimi et al. 2011). Transcriptional regulation is also elicited by negative
  regulators, including NR2E1:ATN1 (atrophin-1) complex, JUN (c-Jun), SNAIL and SLUG
  (Zhang et al. 2006, Vasudevan et al. 2007, Escriva et al. 2008, Uygur et al. 2015)
  and positive regulators such as TP53 (p53), MAF1, ATF2, EGR1 or PPARG (Stambolic
  et al. 2001, Virolle et al. 2001, Patel et al. 2001, Shen et al. 2006, Li et al.
  2016).<p>MicroRNAs miR-26A1, miR-26A2, miR-22, miR-25, miR-302, miR-214, miR-17-5p,
  miR-19 and miR-205 bind PTEN mRNA and inhibit its translation into protein. These
  microRNAs are altered in cancer and can account for changes in PTEN levels (Meng
  et al. 2007, Xiao et al. 2008, Yang et al. 2008, Huse et al. 2009, Kim et al. 2010,
  Poliseno, Salmena, Riccardi et al. 2010, Cai et al. 2013). In addition, coding and
  non-coding RNAs can prevent microRNAs from binding to PTEN mRNA. These RNAs are
  termed competing endogenous RNAs or ceRNAs. Transcripts of the pseudogene PTENP1
  and mRNAs transcribed from SERINC1, VAPA and CNOT6L genes exhibit this activity
  (Poliseno, Salmena, Zhang et al. 2010, Tay et al. 2011, Tay et al. 2014).<p>PTEN
  can translocate from the cytosol to the nucleus after undergoing monoubiquitination.
  PTEN's ability to localize to the nucleus contributes to its tumor suppressive role
  (Trotman et al. 2007). The ubiquitin protease USP7 (HAUSP) targets monoubiquitinated
  PTEN in the nucleus, resulting in PTEN deubiquitination and nuclear exclusion. PML,
  via an unknown mechanism that involves USP7- and PML-interacting protein DAXX, inhibits
  USP7-mediated deubiquitination of PTEN, thus promoting PTEN nuclear localization.
  Disruption of PML function in acute promyelocytic leukemia, through a chromosomal
  translocation that results in expression of a fusion protein PML-RARA, leads to
  aberrant PTEN localization (Song et al. 2008).<p>Several ubiquitin ligases, including
  NEDD4, WWP2, STUB1 (CHIP), RNF146, XIAP and MKRN1, polyubiquitinate PTEN and target
  it for proteasome-mediated degradation (Wang et al. 2007, Van Themsche et al. 2009,
  Ahmed et al. 2011, Maddika et al. 2011, Lee et al. 2015, Li et al. 2015). The ubiquitin
  proteases USP13 and OTUD3, frequently down-regulated in breast cancer, remove polyubiquitin
  chains from PTEN, thus preventing its degradation and increasing its half-life (Zhang
  et al. 2013, Yuan et al. 2015). The catalytic activity of PTEN is negatively regulated
  by PREX2 binding (Fine et al. 2009, Hodakoski et al. 2014) and TRIM27-mediated ubiquitination
  (Lee et al. 2013), most likely through altered PTEN conformation.<p>In addition
  to ubiquitination, PTEN also undergoes SUMOylation (Gonzalez-Santamaria et al. 2012,
  Da Silva Ferrada et al. 2013, Lang et al. 2015, Leslie et al. 2016). SUMOylation
  of the C2 domain of PTEN may regulate PTEN association with the plasma membrane
  (Shenoy et al. 2012) as well as nuclear localization of PTEN (Bassi et al. 2013,
  Collaud et al. 2016). PIASx-alpha, a splicing isorom of E3 SUMO-protein ligase PIAS2
  has been implicated in PTEN SUMOylation (Wang et al. 2014). SUMOylation of PTEN
  may be regulated by activated AKT (Lin et al. 2016). Reactions describing PTEN SUMOylation
  will be annotated when mechanistic details become available.<p>Phosphorylation affects
  the stability and activity of PTEN. FRK tyrosine kinase (RAK) phosphorylates PTEN
  on tyrosine residue Y336, which increases PTEN half-life by inhibiting NEDD4-mediated
  polyubiquitination and subsequent degradation of PTEN. FRK-mediated phosphorylation
  also increases PTEN enzymatic activity (Yim et al. 2009). Casein kinase II (CK2)
  constitutively phosphorylates the C-terminal tail of PTEN on serine and threonine
  residues S370, S380, T382, T383 and S385. CK2-mediated phosphorylation increases
  PTEN protein stability (Torres and Pulido 2001) but results in ~30% reduction in
  PTEN lipid phosphatase activity (Miller et al. 2002).<p>PTEN localization and activity
  are affected by acetylation of its lysine residues (Okumura et al. 2006, Ikenoue
  et al. 2008, Meng et al. 2016). PTEN can undergo oxidation, which affects its function,
  but the mechanism is poorly understood (Tan et al. 2015, Shen et al. 2015, Verrastro
  et al. 2016).  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=6807070
  Reactome].
last-edited: 2021-01-25
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP4047
- /instance/WP4047
revision: null
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP4047.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: PTEN is regulated at the level of gene transcription, mRNA translation,
    localization and protein stability.<p>Transcription of the PTEN gene is regulated
    at multiple levels. Epigenetic repression involves the recruitment of Mi-2/NuRD
    upon SALL4 binding to the PTEN promoter (Yang et al. 2008, Lu et al. 2009) or
    EVI1-mediated recruitment of the polycomb repressor complex (PRC) to the PTEN
    promoter (Song et al. 2009, Yoshimi et al. 2011). Transcriptional regulation is
    also elicited by negative regulators, including NR2E1:ATN1 (atrophin-1) complex,
    JUN (c-Jun), SNAIL and SLUG (Zhang et al. 2006, Vasudevan et al. 2007, Escriva
    et al. 2008, Uygur et al. 2015) and positive regulators such as TP53 (p53), MAF1,
    ATF2, EGR1 or PPARG (Stambolic et al. 2001, Virolle et al. 2001, Patel et al.
    2001, Shen et al. 2006, Li et al. 2016).<p>MicroRNAs miR-26A1, miR-26A2, miR-22,
    miR-25, miR-302, miR-214, miR-17-5p, miR-19 and miR-205 bind PTEN mRNA and inhibit
    its translation into protein. These microRNAs are altered in cancer and can account
    for changes in PTEN levels (Meng et al. 2007, Xiao et al. 2008, Yang et al. 2008,
    Huse et al. 2009, Kim et al. 2010, Poliseno, Salmena, Riccardi et al. 2010, Cai
    et al. 2013). In addition, coding and non-coding RNAs can prevent microRNAs from
    binding to PTEN mRNA. These RNAs are termed competing endogenous RNAs or ceRNAs.
    Transcripts of the pseudogene PTENP1 and mRNAs transcribed from SERINC1, VAPA
    and CNOT6L genes exhibit this activity (Poliseno, Salmena, Zhang et al. 2010,
    Tay et al. 2011, Tay et al. 2014).<p>PTEN can translocate from the cytosol to
    the nucleus after undergoing monoubiquitination. PTEN's ability to localize to
    the nucleus contributes to its tumor suppressive role (Trotman et al. 2007). The
    ubiquitin protease USP7 (HAUSP) targets monoubiquitinated PTEN in the nucleus,
    resulting in PTEN deubiquitination and nuclear exclusion. PML, via an unknown
    mechanism that involves USP7- and PML-interacting protein DAXX, inhibits USP7-mediated
    deubiquitination of PTEN, thus promoting PTEN nuclear localization. Disruption
    of PML function in acute promyelocytic leukemia, through a chromosomal translocation
    that results in expression of a fusion protein PML-RARA, leads to aberrant PTEN
    localization (Song et al. 2008).<p>Several ubiquitin ligases, including NEDD4,
    WWP2, STUB1 (CHIP), RNF146, XIAP and MKRN1, polyubiquitinate PTEN and target it
    for proteasome-mediated degradation (Wang et al. 2007, Van Themsche et al. 2009,
    Ahmed et al. 2011, Maddika et al. 2011, Lee et al. 2015, Li et al. 2015). The
    ubiquitin proteases USP13 and OTUD3, frequently down-regulated in breast cancer,
    remove polyubiquitin chains from PTEN, thus preventing its degradation and increasing
    its half-life (Zhang et al. 2013, Yuan et al. 2015). The catalytic activity of
    PTEN is negatively regulated by PREX2 binding (Fine et al. 2009, Hodakoski et
    al. 2014) and TRIM27-mediated ubiquitination (Lee et al. 2013), most likely through
    altered PTEN conformation.<p>In addition to ubiquitination, PTEN also undergoes
    SUMOylation (Gonzalez-Santamaria et al. 2012, Da Silva Ferrada et al. 2013, Lang
    et al. 2015, Leslie et al. 2016). SUMOylation of the C2 domain of PTEN may regulate
    PTEN association with the plasma membrane (Shenoy et al. 2012) as well as nuclear
    localization of PTEN (Bassi et al. 2013, Collaud et al. 2016). PIASx-alpha, a
    splicing isorom of E3 SUMO-protein ligase PIAS2 has been implicated in PTEN SUMOylation
    (Wang et al. 2014). SUMOylation of PTEN may be regulated by activated AKT (Lin
    et al. 2016). Reactions describing PTEN SUMOylation will be annotated when mechanistic
    details become available.<p>Phosphorylation affects the stability and activity
    of PTEN. FRK tyrosine kinase (RAK) phosphorylates PTEN on tyrosine residue Y336,
    which increases PTEN half-life by inhibiting NEDD4-mediated polyubiquitination
    and subsequent degradation of PTEN. FRK-mediated phosphorylation also increases
    PTEN enzymatic activity (Yim et al. 2009). Casein kinase II (CK2) constitutively
    phosphorylates the C-terminal tail of PTEN on serine and threonine residues S370,
    S380, T382, T383 and S385. CK2-mediated phosphorylation increases PTEN protein
    stability (Torres and Pulido 2001) but results in ~30% reduction in PTEN lipid
    phosphatase activity (Miller et al. 2002).<p>PTEN localization and activity are
    affected by acetylation of its lysine residues (Okumura et al. 2006, Ikenoue et
    al. 2008, Meng et al. 2016). PTEN can undergo oxidation, which affects its function,
    but the mechanism is poorly understood (Tan et al. 2015, Shen et al. 2015, Verrastro
    et al. 2016).  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=6807070
    Reactome].
  keywords:
  - (EZH2) core):PTEN
  - '13(S'')-HODE '
  - 26S proteasome
  - '9S-HODE '
  - ADP
  - 'AGO2 '
  - 'ATN1 '
  - ATP
  - 'Actos '
  - 'BMI1 '
  - 'CBX2 '
  - 'CBX4 '
  - 'CBX6 '
  - 'CBX8 '
  - 'CHD3 '
  - 'CHD4 '
  - CNOT6L mRNA
  - 'CNOT6L mRNA '
  - 'CSNK2A1 '
  - 'CSNK2A2 '
  - 'CSNK2B '
  - Casein kinase II
  - 'EED '
  - EGR1
  - 'EGR1 '
  - EGR1:PTEN gene
  - 'EIF2C1 '
  - 'EIF2C3 '
  - 'EIF2C4 '
  - 'EZH2 '
  - 'GATAD2A '
  - 'GATAD2B '
  - 'GDP '
  - 'GTP '
  - Gene
  - H2O
  - 'HDAC1 '
  - 'HDAC2 '
  - 'HDAC3,5,7 '
  - JUN
  - 'JUN '
  - JUN:PTEN gene
  - K27polyUb-PTEN
  - K48polyUb-K289-PTEN
  - 'K48polyUb-K289-PTEN '
  - K48polyUb-K289-PTEN, PolyUb-K324,K344,K349-RibC-E40,E150,D326-PTEN
  - 'KDM1A '
  - 'LAMTOR1 '
  - 'LAMTOR2 '
  - 'LAMTOR3 '
  - 'LAMTOR4 '
  - 'LAMTOR5 '
  - Ligand
  - Ligand:PTEN gene
  - MAF1
  - 'MAF1 '
  - MAF1:PTEN gene
  - 'MBD3 '
  - MECOM
  - 'MECOM '
  - MECOM:(PRC1.4,PRC2
  - MECOM:PTEN gene
  - MKRN1
  - 'MLST8 '
  - 'MOV10 '
  - 'MTA1 '
  - 'MTA2 '
  - 'MTA3 '
  - 'MTOR '
  - MTORC1:Ragulator:Rag:GNP:RHEB:GTP
  - MonoUb-K13,K289-PTEN
  - NAD+
  - NAM
  - 'NEDD4 '
  - NEDD4,STUB1,WWP2 and
  - 'NR2E1 '
  - NR2E1:(CoREST
  - Nonendonucleolytic
  - NuRD complex
  - 'OTUD3 '
  - 'PHC1 '
  - 'PHC2 '
  - 'PHC3 '
  - PIP3 activates AKT
  - PML
  - 'PPARG '
  - PPARG:Fatty Acid
  - PRC1.4,PRC2 (EZH2)
  - PREX2
  - 'PREX2 '
  - PREX2:PTEN,p-3S,2T-PTEN
  - 'PSMA1 '
  - 'PSMA2 '
  - 'PSMA3 '
  - 'PSMA4 '
  - 'PSMA5 '
  - 'PSMA6 '
  - 'PSMA7 '
  - 'PSMA8 '
  - 'PSMB1 '
  - 'PSMB10 '
  - 'PSMB11 '
  - 'PSMB2 '
  - 'PSMB3 '
  - 'PSMB4 '
  - 'PSMB5 '
  - 'PSMB6 '
  - 'PSMB7 '
  - 'PSMB8 '
  - 'PSMB9 '
  - 'PSMC1 '
  - 'PSMC2 '
  - 'PSMC3 '
  - 'PSMC4 '
  - 'PSMC5 '
  - 'PSMC6 '
  - 'PSMD1 '
  - 'PSMD10 '
  - 'PSMD11 '
  - 'PSMD12 '
  - 'PSMD13 '
  - 'PSMD14 '
  - 'PSMD2 '
  - 'PSMD3 '
  - 'PSMD4 '
  - 'PSMD5 '
  - 'PSMD6 '
  - 'PSMD7 '
  - 'PSMD8 '
  - 'PSMD9 '
  - 'PSME1 '
  - 'PSME2 '
  - 'PSME3 '
  - 'PSME4 '
  - 'PSMF1 '
  - PTEN
  - 'PTEN '
  - PTEN gene
  - 'PTEN gene '
  - PTEN mRNA
  - 'PTEN mRNA '
  - PTEN mRNA:miR-26A
  - PTEN, p-3S,2T-PTEN
  - PTEN:TRIM27
  - PTEN:p-Y387-FRK
  - PTENP1 mRNA
  - 'PTENP1 mRNA '
  - PolyUb-K324,K344,K349-RibC-E40,E150,D326-PTEN
  - 'PolyUb-K324,K344,K349-RibC-E40,E150,D326-PTEN '
  - PolyUb-PTEN
  - 'PolyUb-PTEN '
  - PolyUb-PTEN,
  - PolyUb-PTEN,K48polyUb-K289-PTEN
  - 'RBBP4 '
  - 'RBBP7 '
  - 'RCOR1 '
  - 'REST '
  - 'RGZ '
  - 'RHEB '
  - 'RING1 '
  - RISC
  - RISC:PTEN mRNA
  - RNF146
  - 'RNF2 '
  - 'RPS27A(1-76) '
  - 'RPTOR '
  - 'RRAGA '
  - 'RRAGB '
  - 'RRAGC '
  - 'RRAGD '
  - RibC-E40,E150,D326-PTEN
  - SALL4
  - 'SALL4 '
  - SALL4:NuRD:PTEN gene
  - SALL4:PTEN gene
  - 'SCMH1-2 '
  - 'SHFM1 '
  - 'SLC38A9 '
  - 'SNAI1 '
  - SNAI1,SNAI2
  - SNAI1,SNAI2:PTEN
  - 'SNAI2 '
  - 'STUB1 '
  - 'SUZ12 '
  - 'TNKS '
  - TNKS1/2
  - 'TNKS2 '
  - 'TNRC6A '
  - 'TNRC6B '
  - 'TNRC6C '
  - 'TP53 '
  - TP53 Tetramer
  - TP53 Tetramer:PTEN
  - TRIM27
  - 'TRIM27 '
  - 'UBA52(1-76) '
  - 'UBB(1-76) '
  - 'UBB(153-228) '
  - 'UBB(77-152) '
  - 'UBC(1-76) '
  - 'UBC(153-228) '
  - 'UBC(229-304) '
  - 'UBC(305-380) '
  - 'UBC(381-456) '
  - 'UBC(457-532) '
  - 'UBC(533-608) '
  - 'UBC(609-684) '
  - 'UBC(77-152) '
  - 'USP13 '
  - USP13,OTUD3
  - USP7
  - Ub
  - VAPA mRNA
  - 'VAPA mRNA '
  - 'WWP2 '
  - XIAP
  - 'XIAP '
  - XIAP,NEDD4
  - complex,ATN1,HDAC3,HDAC5,HDAC7)
  - complex,ATN1,HDAC3,HDAC5,HDAC7):PTEN Gene
  - core
  - gene
  - mRNA
  - miR-106 RISC
  - miR-106 RISC:CNOT6L
  - miR-106 RISC:PTEN
  - miR-106 RISC:VAPA
  - 'miR-106a '
  - 'miR-106b '
  - 'miR-17 '
  - miR-17 RISC
  - miR-17 RISC:CNOT6L
  - miR-17 RISC:PTEN
  - miR-17 RISC:VAPA
  - 'miR-19a '
  - miR-19a RISC
  - miR-19a RISC:CNOT6L
  - miR-19a RISC:PTEN
  - miR-19a RISC:VAPA
  - miR-19b RISC
  - miR-19b RISC:CNOT6L
  - miR-19b RISC:PTEN
  - miR-19b RISC:PTENP1
  - 'miR-19b1 '
  - 'miR-19b2 '
  - miR-20 RISC
  - miR-20 RISC:CNOT6L
  - miR-20 RISC:PTEN
  - miR-20 RISC:PTENP1
  - miR-20 RISC:VAPA
  - 'miR-205 '
  - miR-205 RISC
  - miR-205 RISC:PTEN
  - 'miR-20a '
  - 'miR-20b '
  - miR-21
  - 'miR-21 '
  - miR-214
  - 'miR-214 '
  - 'miR-22 '
  - miR-22 RISC
  - miR-22 RISC:PTEN
  - 'miR-25 '
  - miR-25 RISC
  - miR-25 RISC:PTEN
  - miR-26A RISC
  - 'miR-26A1 '
  - 'miR-26A2 '
  - 'miR-93 '
  - miR-93 RISC
  - miR-93:PTEN mRNA
  - p-3S,2T-PTEN
  - 'p-3S,2T-PTEN '
  - p-S109-MKRN1
  - p-S60,S68,S75-MAF1
  - p-T,Y MAPK dimers
  - p-T,p-S-AKT
  - 'p-T185,Y187-MAPK1 '
  - 'p-T202,Y204-MAPK3 '
  - 'p-T305,S472-AKT3 '
  - 'p-T308,S473-AKT1 '
  - 'p-T309,S474-AKT2 '
  - p-T69,T71-ATF2
  - 'p-T69,T71-ATF2 '
  - p-T69,T71-ATF2:PTEN
  - p-Y336-PTEN
  - p-Y387-FRK
  - 'p-Y387-FRK '
  - signaling
  license: CC0
  name: PTEN Regulation
seo: CreativeWork
title: PTEN Regulation
wpid: WP4047
---