---
annotations:
- type: Pathway Ontology
  value: transcription pathway
authors:
- MaintBot
- ReactomeTeam
- Anwesha
- Mkutmon
- Eweitz
description: "<b>OVERVIEW OF TRANSCRIPTION REGULATION:</b>  <br><br>Detailed studies
  of gene transcription regulation in a wide variety of eukaryotic systems has revealed
  the general principles and mechanisms by which cell- or tissue-specific regulation
  of differential gene transcription is mediated (reviewed in Naar, 2001. Kadonaga,
  2004, Maston, 2006, Barolo, 2002; Roeder, 2005, Rosenfeld, 2006).  Of the three
  major classes of DNA polymerase involved in eukaryotic gene transcription, Polymerase
  II generally regulates protein-encoding genes.   Figure 1 shows a diagram of the
  various components involved in cell-specific regulation of Pol-II gene transcription.
  <br><br>Core Promoter: Pol II-regulated genes typically have a Core Promoter where
  Pol II and a variety of general factors bind to specific DNA motifs: <br>   i: the
  TATA box (TATA DNA sequence), which is bound by the \"TATA-binding protein\" (TBP).<br>
  \  ii: the Initiator motif (INR), where Pol II and certain other core factors bind,
  is present in many Pol II-regulated genes.<br>   iii: the Downstream Promoter Element
  (DPE), which is present in a subset of Pol II genes, and where additional core factors
  bind.  <br>The core promoter binding factors are generally ubiquitously expressed,
  although there are exceptions to this.<br><br>Proximal Promoter: immediately upstream
  (5') of the core promoter, Pol II target genes often have a Proximal Promoter region
  that spans up to 500 base pairs (b.p.), or even to 1000 b.p..  This region contains
  a number of functional DNA binding sites for a specific set of transcription activator
  (TA) and transcription repressor (TR) proteins.  These TA and TR factors are generally
  cell- or tissue-specific in expression, rather than ubiquitous, so that the presence
  of their cognate binding sites in the proximal promoter region programs cell- or
  tissue-specific expression of the target gene, perhaps in conjunction with TA and
  TR complexes bound in distal enhancer regions.  <br><br>Distal Enhancer(s): many
  or most Pol II regulated genes in higher eukaryotes have one or more distal Enhancer
  regions which are essential for proper regulation of the gene, often in a cell or
  tissue-specific pattern.  Like the proximal promoter region, each of the distal
  enhancer regions typically contain a cluster of binding sites for specific TA and/or
  TR DNA-binding factors, rather than just a single site. <br><br>   Enhancers generally
  have three defining characteristics:<br>     i: They can be located very long distances
  from the promoter of the target gene they regulate, sometimes as far as 100 Kb,
  or more.<br>     ii: They can be either upstream (5') or downstream (3') of the
  target gene, including within introns of that gene.<br>     iii: They can function
  in either orientation in the DNA.<br><br>Combinatorial mechanisms of transcription
  regulation: The specific combination of TA and TR binding sites within the proximal
  promoter and/or distal enhancer(s) provides a \"combinatorial transcription code\"
  that mediates cell- or tissue-specific expression of the associated target gene.
  \ Each promoter or enhancer region mediates expression in a specific subset of the
  overall expression pattern.  In at least some cases, each enhancer region functions
  completely independently of the others, so that the overall expression pattern is
  a linear combination of the expression patterns of each of the enhancer modules.<br><br>Co-Activator
  and Co-Repressor Complexes:  DNA-bound TA and TR proteins typically recruit the
  assembly of specific Co-Activator (Co-A) and Co-Repressor (Co-R) Complexes, respectively,
  which are essential for regulating target gene transcription.  Both Co-A's and Co-R's
  \ are multi-protein complexes that contain several specific protein components.<br><br>Co-Activator
  complexes generally contain at lease one component protein that has Histone Acetyl
  Transferase (HAT) enzymatic activity.  This functions to acetylate Histones and/or
  other chromatin-associated factors, which typically increases that transcription
  activation of the target gene.  By contrast, Co-Repressor complexes generally contain
  at lease one component protein that has Histone De-Acetylase (HDAC) enzymatic activity.
  \ This functions to de-acetylate Histones and/or other chromatin-associated factors.
  This typically increases the transcription repression of the target gene.<br><br>Adaptor
  (Mediator) complexes: In addition to the co-activator complexes that assemble on
  particular cell-specific TA factors,  - there are at least two additional transcriptional
  co-activator complexes common to most cells.  One of these is the Mediator complex,
  which functions as an \"adaptor\" complex that bridges between the tissue-specific
  co-activator complexes assembled in the proximal promoter (or distal enhancers).
  \ The human Mediator complex has been shown to contain at least 19 protein distinct
  components.  Different combinations of these co-activator proteins are also found
  to be components of specific transcription Co-Activator complexes, such as the DRIP,
  TRAP and ARC complexes described below.  <br><br>TBP/TAF complex: Another large
  Co-A complex is the \"TBP-associated factors\" (TAFs) that assemble on TBP (TATA-Binding
  Protein), which is bound to the TATA box present in many promoters.  There are at
  least 23 human TAF proteins that have been identified.  Many of these are ubiquitously
  expressed, but TAFs can also be expressed in a cell or tissue-specific pattern.
  \ <br><br>   <b>\tSpecific Coactivator Complexes for DNA-binding Transcription Factors.</b>
  \  <br><br>A number of specific co-activator complexes for DNA-binding transcription
  factors have been identified, including DRIP, TRAP, and ARC (reviewed in Bourbon,
  2004, Blazek, 2005, Conaway, 2005, and Malik, 2005).  The DRIP co-activator complex
  was originally identified and named as a specific complex associated with the Vitamin
  D Receptor member of the nuclear receptor family of transcription factors (Rachez,
  1998).  Similarly, the TRAP co-activator complex was originally identified as a
  complex that associates with the thyroid receptor (Yuan, 1998).   It was later determined
  that all of the components of the DRIP complex are also present in the TRAP complex,
  and the ARC complex (discussed further below).  For example, the DRIP205 and TRAP220
  proteins were show to be identical, as were specific pairs of the other components
  of these complexes (Rachez, 1999).<br><br>In addition, these various transcription
  co-activator proteins identified in mammalian cells were found to be the orthologues
  or homologues of the Mediator (\"adaptor\") complex proteins (reviewed in Bourbon,
  2004).  The Mediator proteins were originally identified in yeast  by Kornberg and
  colleagues, as complexes associated with DNA polymerase (Kelleher, 1990).  In higher
  organisms, Adapter complexes bridge between the basal transcription factors (including
  Pol II) and tissue-specific transcription factors (TFs) bound to sites within upstream
  Proximal Promoter regions or distal Enhancer regions (Figure 1).  However, many
  of the Mediator homologues can also be found in complexes associated with specific
  transcription factors in higher organisms.  A unified nomenclature system for these
  adapter / co-activator proteins now labels them Mediator 1 through Mediator 31 (Bourbon,
  2004).  For example, the DRIP205 / TRAP220 proteins are now identified as Mediator
  1 (Rachez, 1999), based on homology with yeast Mediator 1.<br><br>    <b>Example
  Pathway: Specific Regulation of Target Genes During Notch Signaling:</b> <br><br>One
  well-studied example of cell-specific regulation of gene transcription is selective
  regulation of target genes during Notch signaling.  Notch signaling was first identified
  in Drosophila, where it has been studied in detail at the genetic, molecular, biochemical
  and cellular levels (reviewed in Justice, 2002; Bray, 2006; Schweisguth, 2004; Louvri,
  2006).  In Drosophila, Notch signaling to the nucleus is thought always to be mediated
  by one specific DNA binding transcription factor, Suppressor of Hairless.  In mammals,
  the homologous genes are called CBF1 (or RBPJkappa), while in worms they are called
  Lag-1, so that the acronym \"CSL\" has been given to this conserved transcription
  factor family.  There are at least two human CSL homologues, which are now named
  RBPJ and RBPJL.  <br><br>In Drosophila, Su(H) is known to be bifunctional, in that
  it represses target gene transcription in the absence of Notch signaling, but activates
  target genes during Notch signaling.  At least some of the mammalian CSL homologues
  are believed also to be bifunctional, and to mediate target gene repression in the
  absence of Notch signaling, and activation in the presence of Notch signaling.<br><br>Notch
  Co-Activator and Co-Repressor complexes: This repression is mediated by at least
  one specific co-repressor complexes (Co-R) bound to CSL in the absence of Notch
  signaling.  In Drosophila, this co-repressor complex consists of at least three
  distinct co-repressor proteins: Hairless, Groucho, and dCtBP (Drosophila C-terminal
  Binding Protein).  Hairless has been show to bind directly to Su(H), and Groucho
  and dCtBP have been shown to bind directly to Hairless (Barolo, 2002).  All three
  of the co-repressor proteins have been shown to be necessary for proper gene regulation
  during Notch signaling in vivo (Nagel, 2005).<br><br>In mammals, the same general
  pathway and mechanisms are observed, where CSL proteins are bifunctional DNA binding
  transcription factors (TFs), that bind to Co-Repressor complexes to mediate repression
  in the absence of Notch signaling, and bind to Co-Activator  complexes to mediate
  activation in the presence of Notch signaling.  However, in mammals, there may be
  multiple co-repressor complexes, rather than the single Hairless co-repressor complex
  that has been observed in Drosophila.  <br><br>During Notch signaling in all systems,
  the Notch transmembrane receptor is cleaved and the Notch intracellular domain (NICD)
  translocates to the nucleus, where it there functions as a specific transcription
  co-activator for CSL proteins.  In the nucleus, NICD replaces the Co-R complex bound
  to CSL, thus resulting in de-repression of  Notch target genes in the nucleus (Figure
  2).  Once bound to CSL, NICD and CSL proteins recruit an additional co-activator
  protein, Mastermind, to form a CSL-NICD-Mam ternary co-activator (Co-A) complex.
  \ This Co-R  complex was initially thought to be sufficient to mediate activation
  of  at least some Notch target genes.  However, there now is evidence that still
  other co-activators and additional DNA-binding transcription factors are required
  in at least some contexts (reviewed in Barolo, 2002).  <br><br>Thus, CSL is a good
  example of a bifunctional DNA-binding transcription factor that mediates repression
  of specific targets genes in one context, but activation of the same targets in
  another context.  This bifunctionality is mediated by the association of specific
  Co-Repressor complexes vs. specific Co-Activator complexes in different contexts,
  namely in the absence or presence of Notch signaling.  View original pathway at
  [http://www.reactome.org/PathwayBrowser/#DIAGRAM=212436 Reactome]."
last-edited: 2021-05-22
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP1822
- /instance/WP1822
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP1822.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: "<b>OVERVIEW OF TRANSCRIPTION REGULATION:</b>  <br><br>Detailed studies
    of gene transcription regulation in a wide variety of eukaryotic systems has revealed
    the general principles and mechanisms by which cell- or tissue-specific regulation
    of differential gene transcription is mediated (reviewed in Naar, 2001. Kadonaga,
    2004, Maston, 2006, Barolo, 2002; Roeder, 2005, Rosenfeld, 2006).  Of the three
    major classes of DNA polymerase involved in eukaryotic gene transcription, Polymerase
    II generally regulates protein-encoding genes.   Figure 1 shows a diagram of the
    various components involved in cell-specific regulation of Pol-II gene transcription.
    <br><br>Core Promoter: Pol II-regulated genes typically have a Core Promoter where
    Pol II and a variety of general factors bind to specific DNA motifs: <br>   i:
    the TATA box (TATA DNA sequence), which is bound by the \"TATA-binding protein\"
    (TBP).<br>   ii: the Initiator motif (INR), where Pol II and certain other core
    factors bind, is present in many Pol II-regulated genes.<br>   iii: the Downstream
    Promoter Element (DPE), which is present in a subset of Pol II genes, and where
    additional core factors bind.  <br>The core promoter binding factors are generally
    ubiquitously expressed, although there are exceptions to this.<br><br>Proximal
    Promoter: immediately upstream (5') of the core promoter, Pol II target genes
    often have a Proximal Promoter region that spans up to 500 base pairs (b.p.),
    or even to 1000 b.p..  This region contains a number of functional DNA binding
    sites for a specific set of transcription activator (TA) and transcription repressor
    (TR) proteins.  These TA and TR factors are generally cell- or tissue-specific
    in expression, rather than ubiquitous, so that the presence of their cognate binding
    sites in the proximal promoter region programs cell- or tissue-specific expression
    of the target gene, perhaps in conjunction with TA and TR complexes bound in distal
    enhancer regions.  <br><br>Distal Enhancer(s): many or most Pol II regulated genes
    in higher eukaryotes have one or more distal Enhancer regions which are essential
    for proper regulation of the gene, often in a cell or tissue-specific pattern.
    \ Like the proximal promoter region, each of the distal enhancer regions typically
    contain a cluster of binding sites for specific TA and/or TR DNA-binding factors,
    rather than just a single site. <br><br>   Enhancers generally have three defining
    characteristics:<br>     i: They can be located very long distances from the promoter
    of the target gene they regulate, sometimes as far as 100 Kb, or more.<br>     ii:
    They can be either upstream (5') or downstream (3') of the target gene, including
    within introns of that gene.<br>     iii: They can function in either orientation
    in the DNA.<br><br>Combinatorial mechanisms of transcription regulation: The specific
    combination of TA and TR binding sites within the proximal promoter and/or distal
    enhancer(s) provides a \"combinatorial transcription code\" that mediates cell-
    or tissue-specific expression of the associated target gene.  Each promoter or
    enhancer region mediates expression in a specific subset of the overall expression
    pattern.  In at least some cases, each enhancer region functions completely independently
    of the others, so that the overall expression pattern is a linear combination
    of the expression patterns of each of the enhancer modules.<br><br>Co-Activator
    and Co-Repressor Complexes:  DNA-bound TA and TR proteins typically recruit the
    assembly of specific Co-Activator (Co-A) and Co-Repressor (Co-R) Complexes, respectively,
    which are essential for regulating target gene transcription.  Both Co-A's and
    Co-R's  are multi-protein complexes that contain several specific protein components.<br><br>Co-Activator
    complexes generally contain at lease one component protein that has Histone Acetyl
    Transferase (HAT) enzymatic activity.  This functions to acetylate Histones and/or
    other chromatin-associated factors, which typically increases that transcription
    activation of the target gene.  By contrast, Co-Repressor complexes generally
    contain at lease one component protein that has Histone De-Acetylase (HDAC) enzymatic
    activity.  This functions to de-acetylate Histones and/or other chromatin-associated
    factors. This typically increases the transcription repression of the target gene.<br><br>Adaptor
    (Mediator) complexes: In addition to the co-activator complexes that assemble
    on particular cell-specific TA factors,  - there are at least two additional transcriptional
    co-activator complexes common to most cells.  One of these is the Mediator complex,
    which functions as an \"adaptor\" complex that bridges between the tissue-specific
    co-activator complexes assembled in the proximal promoter (or distal enhancers).
    \ The human Mediator complex has been shown to contain at least 19 protein distinct
    components.  Different combinations of these co-activator proteins are also found
    to be components of specific transcription Co-Activator complexes, such as the
    DRIP, TRAP and ARC complexes described below.  <br><br>TBP/TAF complex: Another
    large Co-A complex is the \"TBP-associated factors\" (TAFs) that assemble on TBP
    (TATA-Binding Protein), which is bound to the TATA box present in many promoters.
    \ There are at least 23 human TAF proteins that have been identified.  Many of
    these are ubiquitously expressed, but TAFs can also be expressed in a cell or
    tissue-specific pattern.  <br><br>   <b>\tSpecific Coactivator Complexes for DNA-binding
    Transcription Factors.</b>   <br><br>A number of specific co-activator complexes
    for DNA-binding transcription factors have been identified, including DRIP, TRAP,
    and ARC (reviewed in Bourbon, 2004, Blazek, 2005, Conaway, 2005, and Malik, 2005).
    \ The DRIP co-activator complex was originally identified and named as a specific
    complex associated with the Vitamin D Receptor member of the nuclear receptor
    family of transcription factors (Rachez, 1998).  Similarly, the TRAP co-activator
    complex was originally identified as a complex that associates with the thyroid
    receptor (Yuan, 1998).   It was later determined that all of the components of
    the DRIP complex are also present in the TRAP complex, and the ARC complex (discussed
    further below).  For example, the DRIP205 and TRAP220 proteins were show to be
    identical, as were specific pairs of the other components of these complexes (Rachez,
    1999).<br><br>In addition, these various transcription co-activator proteins identified
    in mammalian cells were found to be the orthologues or homologues of the Mediator
    (\"adaptor\") complex proteins (reviewed in Bourbon, 2004).  The Mediator proteins
    were originally identified in yeast  by Kornberg and colleagues, as complexes
    associated with DNA polymerase (Kelleher, 1990).  In higher organisms, Adapter
    complexes bridge between the basal transcription factors (including Pol II) and
    tissue-specific transcription factors (TFs) bound to sites within upstream Proximal
    Promoter regions or distal Enhancer regions (Figure 1).  However, many of the
    Mediator homologues can also be found in complexes associated with specific transcription
    factors in higher organisms.  A unified nomenclature system for these adapter
    / co-activator proteins now labels them Mediator 1 through Mediator 31 (Bourbon,
    2004).  For example, the DRIP205 / TRAP220 proteins are now identified as Mediator
    1 (Rachez, 1999), based on homology with yeast Mediator 1.<br><br>    <b>Example
    Pathway: Specific Regulation of Target Genes During Notch Signaling:</b> <br><br>One
    well-studied example of cell-specific regulation of gene transcription is selective
    regulation of target genes during Notch signaling.  Notch signaling was first
    identified in Drosophila, where it has been studied in detail at the genetic,
    molecular, biochemical and cellular levels (reviewed in Justice, 2002; Bray, 2006;
    Schweisguth, 2004; Louvri, 2006).  In Drosophila, Notch signaling to the nucleus
    is thought always to be mediated by one specific DNA binding transcription factor,
    Suppressor of Hairless.  In mammals, the homologous genes are called CBF1 (or
    RBPJkappa), while in worms they are called Lag-1, so that the acronym \"CSL\"
    has been given to this conserved transcription factor family.  There are at least
    two human CSL homologues, which are now named RBPJ and RBPJL.  <br><br>In Drosophila,
    Su(H) is known to be bifunctional, in that it represses target gene transcription
    in the absence of Notch signaling, but activates target genes during Notch signaling.
    \ At least some of the mammalian CSL homologues are believed also to be bifunctional,
    and to mediate target gene repression in the absence of Notch signaling, and activation
    in the presence of Notch signaling.<br><br>Notch Co-Activator and Co-Repressor
    complexes: This repression is mediated by at least one specific co-repressor complexes
    (Co-R) bound to CSL in the absence of Notch signaling.  In Drosophila, this co-repressor
    complex consists of at least three distinct co-repressor proteins: Hairless, Groucho,
    and dCtBP (Drosophila C-terminal Binding Protein).  Hairless has been show to
    bind directly to Su(H), and Groucho and dCtBP have been shown to bind directly
    to Hairless (Barolo, 2002).  All three of the co-repressor proteins have been
    shown to be necessary for proper gene regulation during Notch signaling in vivo
    (Nagel, 2005).<br><br>In mammals, the same general pathway and mechanisms are
    observed, where CSL proteins are bifunctional DNA binding transcription factors
    (TFs), that bind to Co-Repressor complexes to mediate repression in the absence
    of Notch signaling, and bind to Co-Activator  complexes to mediate activation
    in the presence of Notch signaling.  However, in mammals, there may be multiple
    co-repressor complexes, rather than the single Hairless co-repressor complex that
    has been observed in Drosophila.  <br><br>During Notch signaling in all systems,
    the Notch transmembrane receptor is cleaved and the Notch intracellular domain
    (NICD) translocates to the nucleus, where it there functions as a specific transcription
    co-activator for CSL proteins.  In the nucleus, NICD replaces the Co-R complex
    bound to CSL, thus resulting in de-repression of  Notch target genes in the nucleus
    (Figure 2).  Once bound to CSL, NICD and CSL proteins recruit an additional co-activator
    protein, Mastermind, to form a CSL-NICD-Mam ternary co-activator (Co-A) complex.
    \ This Co-R  complex was initially thought to be sufficient to mediate activation
    of  at least some Notch target genes.  However, there now is evidence that still
    other co-activators and additional DNA-binding transcription factors are required
    in at least some contexts (reviewed in Barolo, 2002).  <br><br>Thus, CSL is a
    good example of a bifunctional DNA-binding transcription factor that mediates
    repression of specific targets genes in one context, but activation of the same
    targets in another context.  This bifunctionality is mediated by the association
    of specific Co-Repressor complexes vs. specific Co-Activator complexes in different
    contexts, namely in the absence or presence of Notch signaling.  View original
    pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=212436 Reactome]."
  keywords:
  - 'ZNF490 '
  - 'ZNF750 '
  - 'ZNF195 '
  - 'NR5A2-2 '
  - RBPJ:NCOR
  - 'ZNF670 '
  - 'ZNF253 '
  - MED17
  - 'HDAC3 '
  - regulation by RUNX3
  - 'ZNF440 '
  - 'ZNF75D '
  - 'ZNF764 '
  - 'ZNF213 '
  - 'ZNF45 '
  - 'ZNF43 '
  - 'ZNF773 '
  - 'ZNF180 '
  - 'ZNF554 '
  - 'ZNF620 '
  - 'NR1I2-1 '
  - 'NR1D2 '
  - 'ZNF775 '
  - 'ZNF496 '
  - 'ZNF793 '
  - 'ZNF772 '
  - 'ZNF333 '
  - 'ZNF433 '
  - 'ZNF479 '
  - 'NR1I2-6 '
  - MED30
  - 'ZNF710 '
  - 'ZNF791 '
  - 'ZNF230 '
  - 'NR0B2 '
  - 'ZNF721 '
  - 'ZNF790 '
  - 'TRIM28 '
  - 'ZNF454 '
  - 'ZNF705A '
  - 'ZNF439 '
  - 'NRBF2-1 '
  - 'ZNF676 '
  - 'NICD4 '
  - 'ZNF10 '
  - 'RARA '
  - 'NICD1 '
  - 'PGR-2 '
  - 'ZNF500 '
  - 'NR3C2-3 '
  - 'ZNF692 '
  - 'ZNF704 '
  - 'ZNF559 '
  - SMAD2/SMAD3:SMAD4
  - 'ZNF555 '
  - 'PPARA '
  - 'SNW1 '
  - 'MED10 '
  - 'MAML1 '
  - KRAB-ZNF
  - 'ZNF691 '
  - 'HDAC1 '
  - 'NR2E3-2 '
  - 'NR1H2 '
  - 'ZNF430 '
  - 'NR1H3-1 '
  - 'ZFP2 '
  - 'ZNF273 '
  - 'ZNF493 '
  - 'ZKSCAN7 '
  - 'ZNF658 '
  - 'ZNF565 '
  - 'ZNF460 '
  - NR-MED1 Coactivator
  - 'ZNF266 '
  - 'ZNF254 '
  - 'ZNF619 '
  - 'ZNF568 '
  - 'ZNF155 '
  - 'CREBBP '
  - 'ZNF485 '
  - 'PPARG '
  - 'ZNF714 '
  - KAP (KRAB-Domain
  - 'ZNF274 '
  - 'NR3C1-2 '
  - 'ZNF610 '
  - 'ZNF586 '
  - 'ZNF681 '
  - 'NR3C1-7 '
  - 'NR5A2-1 '
  - 'ZNF74 '
  - 'ZNF324B '
  - 'ZNF425 '
  - 'ZNF606 '
  - 'RORB '
  - 'ZNF311 '
  - gene expression
  - 'ZNF492 '
  - 'ZNF786 '
  - 'ZKSCAN5 '
  - 'NR1I2-4 '
  - 'ZNF432 '
  - 'NR4A2 '
  - 'CDK8 '
  - MED7
  - 'ZNF605 '
  - 'ZNF208 '
  - 'KAT2A '
  - 'ZNF662 '
  - 'ZNF14 '
  - 'HDAC10 '
  - 'CCNC '
  - 'NR6A1-1 '
  - 'ZNF567 '
  - 'ZNF839 '
  - MED20
  - transcription
  - 'ZNF563 '
  - 'ZNF350 '
  - 'ZNF726 '
  - corepressor complex
  - 'ZKSCAN1 '
  - 'ZNF343 '
  - 'NR1D1 '
  - 'ZNF517 '
  - 'ZIM2 '
  - 'NR3C2-4 '
  - RBPJ
  - 'HDAC11 '
  - 'ZNF589 '
  - 'ZNF707 '
  - 'ZNF583 '
  - 'ZNF189 '
  - 'ZNF613 '
  - 'ZNF716 '
  - 'HDAC7 '
  - MED31
  - 'ZNF484 '
  - 'ZFP69 '
  - 'ZNF175 '
  - 'ZNF530 '
  - 'NR3C1-3 '
  - 'ZNF667 '
  - 'ZNF528 '
  - 'ZNF736 '
  - 'ZNF627 '
  - 'ZNF557 '
  - 'ZNF70 '
  - 'ZNF732 '
  - 'ZNF547 '
  - 'ZNF761 '
  - 'ZNF566 '
  - MED25
  - 'ZNF184 '
  - 'ZNF25 '
  - FOXO-mediated
  - 'TBL1XR1 '
  - 'ZNF223 '
  - 'ZNF354B '
  - 'MED4 '
  - 'ZNF671 '
  - Regulation by E2F6
  - 'ZNF724P '
  - 'ZNF17 '
  - 'ZNF256 '
  - 'ZNF529 '
  - 'ZNF726P1 '
  - 'ZNF777 '
  - Regulation by VENTX
  - 'ZNF615 '
  - 'ZNF37A '
  - regulation by the
  - complex
  - 'ZNF697 '
  - 'ZNF746 '
  - CSL NICD coactivator
  - 'ZNF706 '
  - 'MED7 '
  - 'ZNF468 '
  - 'MED20 '
  - 'ZNF561 '
  - 'NICD2 '
  - 'ZNF711 '
  - 'NR4A3-1 '
  - 'ZNF442 '
  - 'ZFP1 '
  - 'ZNF226 '
  - 'ZNF431 '
  - 'ZNF26 '
  - 'ZNF792 '
  - 'MED17 '
  - 'PPARD '
  - 'ZNF506 '
  - MED14
  - 'ZNF544 '
  - 'MED8 '
  - 'ZNF75A '
  - 'VDR '
  - 'ZNF626 '
  - 'ZNF227 '
  - 'ZNF473 '
  - 'ZNF778 '
  - 'ZNF705D '
  - 'ZNF287 '
  - 'ZNF257(1-535) '
  - 'ZNF264 '
  - 'ZNF100 '
  - 'ZNF23 '
  - TRAP coactivator
  - 'ZNF337 '
  - 'ZNF684 '
  - 'ZNF304 '
  - PCAF
  - 'ZNF660 '
  - 'RARB '
  - 'RXRB '
  - 'NR6A1-2 '
  - 'ZNF441 '
  - 'ZSCAN25 '
  - 'ZNF678 '
  - 'NR4A3-2 '
  - 'ZNF689 '
  - 'NR1H4-2 '
  - 'NR2F6 '
  - NR
  - 'ZNF234 '
  - 'ZIM3 '
  - 'ZNF616 '
  - 'ZNF564 '
  - 'ZNF585A '
  - 'ZNF302 '
  - 'ZNF705E '
  - 'ZNF33A '
  - 'ZNF569 '
  - 'RBPJ '
  - 'ZNF20 '
  - 'ZNF703 '
  - 'ZNF543 '
  - DRIP coactivator
  - MED16
  - 'ZNF12(1-501) '
  - 'ZNF641 '
  - NR-NCOR CoRepressor
  - 'ZNF624 '
  - 'ZNF771 '
  - MED10
  - 'ZNF382 '
  - 'ZNF804B '
  - 'ZNF549 '
  - 'ZNF770 '
  - 'ZNF486 '
  - 'ZNF426 '
  - 'ZNF114 '
  - 'ZNF840 '
  - 'NR3C1-4 '
  - 'ZNF625 '
  - 'MED23 '
  - 'NR3C1-1 '
  - 'ZNF436 '
  - 'NR1I2-7 '
  - 'HDAC4 '
  - 'AR '
  - NCOR1, NCOR2
  - 'ZNF548 '
  - (TAZ)-stimulated
  - 'ZFP28 '
  - 'HKR1 '
  - 'NR2C2AP '
  - YAP1- and WWTR1
  - 'ZNF720 '
  - 'NR3C1-5 '
  - 'ZNF470 '
  - 'ZNF614 '
  - 'ZNF415 '
  - MED6
  - 'ZNF727 '
  - 'KRBA1 '
  - 'TBL1X '
  - 'ZNF394 '
  - MED4
  - 'NRBF2-2 '
  - 'NR6A1-5 '
  - 'ZNF718 '
  - 'ZNF675 '
  - 'NR2E3-1 '
  - 'ZNF285 '
  - MED1
  - 'THRA '
  - 'ZNF570 '
  - 'ZNF664 '
  - 'ZNF141 '
  - 'MAML3 '
  - 'NR1I2-3 '
  - 'ZNF445 '
  - 'ZNF546 '
  - 'NR3C2-1 '
  - 'MAML2 '
  - AP-2 (TFAP2) family
  - 'ZNF202 '
  - 'MED24 '
  - 'ZNF747 '
  - 'ZNF597 '
  - ARC coactivator
  - 'ZNF749 '
  - 'ZFP90 '
  - 'ZNF250 '
  - 'ZNF730 '
  - 'MED12 '
  - 'ZNF300 '
  - 'ESRRG '
  - 'ZNF18 '
  - 'MED30 '
  - 'HDAC6 '
  - 'MED6 '
  - 'ZNF702P '
  - 'ESR1 '
  - 'NR3C1-9 '
  - CDK8
  - Complex
  - 'ZNF79 '
  - 'ZNF587 '
  - activity of
  - 'ZNF225 '
  - 'ZNF514 '
  - 'ZNF595 '
  - 'HNF4A '
  - MED24
  - 'ESRRA '
  - 'ZNF154 '
  - 'ZNF354A '
  - 'ZNF677 '
  - of transcription
  - 'ZNF101 '
  - 'ZNF611 '
  - 'ZNF33B '
  - 'ZNF19 '
  - 'ZNF3 '
  - 'NR5A1 '
  - CREBBP
  - 'ZNF709 '
  - 'ZNF248 '
  - 'RORC '
  - MED27
  - 'ZNF682 '
  - 'ZNF221 '
  - 'ZNF282 '
  - 'RXRG '
  - 'ZNF701 '
  - 'ZNF571 '
  - 'ZKSCAN4 '
  - 'MED14 '
  - 'ZNF2 '
  - 'ZNF317 '
  - 'ZNF200 '
  - 'ZIK1 '
  - 'ZNF383 '
  - 'RXRA '
  - 'NR0B1-2 '
  - CCNC
  - TBL1
  - 'ZNF224 '
  - 'ZNF320 '
  - 'ZNF334 '
  - 'NR2C1 '
  - 'ZNF30 '
  - 'ZNF584 '
  - 'ESR2 '
  - 'ZNF222 '
  - 'ZNF133 '
  - 'RARG '
  - 'ZNF708 '
  - 'NR1H3-2 '
  - 'ZNF136 '
  - 'NR1I2-5 '
  - heterotrimer
  - 'ZNF785 '
  - 'ZNF99 '
  - 'ZNF679 '
  - 'NR1I3-2 '
  - 'ZNF774 '
  - 'NR4A1 '
  - 'ZNF740 '
  - 'NR1I3-1 '
  - 'ZNF607 '
  - 'ZNF157 '
  - 'ZNF169 '
  - 'ZNF416 '
  - 'MED1 '
  - 'ZNF443 '
  - 'ZFP14 '
  - 'NR1H4-4 '
  - 'THRB '
  - 'ZNF573 '
  - 'NR1I2-2 '
  - 'ZNF71 '
  - 'ZNF556 '
  - 'ESRRB '
  - 'ZNF124 '
  - MED15
  - 'NICD3 '
  - 'ZNF729 '
  - 'ZNF215 '
  - 'ZNF582 '
  - 'ZNF354C '
  - 'MED31 '
  - 'ZSCAN32 '
  - 'ZNF767 '
  - 'KAT2B '
  - 'ZNF160 '
  - 'ZKSCAN8 '
  - 'ZNF649 '
  - HDAC
  - 'HDAC9 '
  - NICD
  - 'ZNF665 '
  - 'ZNF286A '
  - 'ZNF263 '
  - 'ZNF205 '
  - 'NR3C1-6 '
  - 'NR3C2-2 '
  - 'ZFP30 '
  - 'ZNF540 '
  - 'HDAC2 '
  - 'ZNF480 '
  - MED12
  - 'ZNF782 '
  - 'NR6A1-4 '
  - 'NR1H4-1 '
  - 'ZNF419 '
  - 'ZNF41 '
  - 'ZNF446 '
  - 'ZNF551 '
  - 'ZNF233 '
  - Transcriptional
  - regulation by RUNX1
  - 'NCOR2 '
  - 'ZNF696 '
  - 'ZNF558 '
  - 'ZNF331 '
  - 'HNF4G '
  - 'ZNF483 '
  - 'ZNF776 '
  - 'ZNF562 '
  - 'ZNF680 '
  - 'MED15 '
  - Regulation by MECP2
  - 'MED26 '
  - 'HDAC8 '
  - 'ZNF599 '
  - 'NR1H4-3 '
  - 'ZFP69B '
  - 'ZNF519 '
  - SNW1
  - factors
  - Regulation by TP53
  - 'NR0B1-1 '
  - Associated Protein)
  - 'ZNF214 '
  - 'ZNF669 '
  - regulation by RUNX2
  - 'ZNF799 '
  - 'ZNF235 '
  - 'ZNF135 '
  - 'ZNF461 '
  - 'ZKSCAN3 '
  - 'NCOR1 '
  - 'ZNF324 '
  - 'ZNF112 '
  - KRAB-ZNF / KAP
  - 'ZNF738 '
  - 'ZNF510 '
  - 'ZNF347 '
  - 'NR2F1 '
  - 'ZNF552 '
  - 'ZNF560 '
  - 'MED27 '
  - 'PRDM7 '
  - 'ZNF668 '
  - 'MED16 '
  - 'ZNF705G '
  - 'ZNF737 '
  - MED8
  - 'ZNF268 '
  - 'ZNF34 '
  - 'ZNF550 '
  - 'NR2C2 '
  - 'NR2E1 '
  - 'ZNF398 '
  - 'ZFP37 '
  - 'ZNF655 '
  - 'ZNF417 '
  - 'NR6A1-3 '
  - MED26
  - 'ZNF600 '
  - 'MED13 '
  - 'ZNF92 '
  - 'ZNF621 '
  - 'ZNF700 '
  - 'ZNF28 '
  - 'ZNF140 '
  - 'ZNF699 '
  - 'MAMLD1 '
  - 'MED25 '
  - 'ZNF688 '
  - 'ZNF420 '
  - 'ZNF138 '
  - 'ZNF713 '
  - 'NR3C1-8 '
  - 'PGR '
  - 'KRBOX4 '
  - 'ZNF585B '
  - 'ZNF735 '
  - MAML
  - 'ZNF429 '
  - MED13
  - 'ZNF77 '
  - 'NRBP1 '
  - 'ZNF577 '
  - MED23
  - 'ZNF658B '
  - 'NR5A2-3 '
  - 'ZNF596 '
  - 'ZNF211 '
  - 'HDAC5 '
  - 'ZNF705F '
  - 'ZNF418 '
  - 'ZNF212 '
  - 'ZNF197 '
  - 'ZNF717 '
  - 'ZNF860 '
  - 'ZNF471 '
  - 'RORA '
  - 'ZNF75CP '
  - 'ZNF267 '
  license: CC0
  name: Generic transcription pathway
seo: CreativeWork
title: Generic transcription pathway
wpid: WP1822
---