---
annotations:
- id: PW:0000004
  parent: regulatory pathway
  type: Pathway Ontology
  value: regulatory pathway
- id: PW:0000099
  parent: regulatory pathway
  type: Pathway Ontology
  value: DNA repair pathway
authors:
- ReactomeTeam
- Ryanmiller
description: 'The mismatch repair (MMR) system corrects single base mismatches and
  small insertion and deletion loops (IDLs) of unpaired bases. MMR is primarily associated
  with DNA replication and is highly conserved across prokaryotes and eukaryotes.
  MMR consists of the following basic steps: a sensor (MutS homologue) detects a mismatch
  or IDL, the sensor activates a set of proteins (a MutL homologue and an exonuclease)
  that select the nascent DNA strand to be repaired, nick the strand, exonucleolytically
  remove  a region of nucleotides containing the mismatch, and finally a DNA polymerase
  resynthesizes the strand and a ligase seals the remaining nick (reviewed in Kolodner
  and Marsischkny 1999, Iyer et al. 2006, Li 2008, Fukui 2010, Jiricny 2013). <br>Humans
  have 2 different MutS complexes. The MSH2:MSH6 heterodimer (MutSalpha) recognizes
  single base mismatches and small loops of one or two unpaired bases. The MSH2:MSH3
  heterodimer (MutSbeta) recognizes loops of two or more unpaired bases. Upon binding
  a mismatch, the MutS complex becomes activated in an ATP-dependent manner allowing
  for subsequent downstream interactions and movement on the DNA substrate. (There
  are two mechanisms proposed: a sliding clamp and a switch diffusion model.) Though
  the order of steps and structural details are not fully known, the activated MutS
  complex interacts with MLH1:PMS2 (MutLalpha) and PCNA, the sliding clamp present
  at replication foci. The role of PCNA is multifaceted as it may act as a processivity
  factor in recruiting MMR proteins to replicating DNA, interact with MLH1:PMS2 and
  Exonuclease 1 (EXO1) to initiate excision of the recently replicated strand and
  direct DNA polymerase delta to initiate replacement of bases. MLH1:PMS2 makes an
  incision in the strand to be repaired and EXO1 extends the incision to make a single-stranded
  gap of up to 1 kb that removes the mismatched base(s). (Based on assays of purified
  human proteins, there is also a variant of the mismatch repair pathway that does
  not require EXO1, however the mechanism is not clear. EXO1 is almost always required,
  it is possible that the exonuclease activity of DNA polymerase delta may compensate
  in some situations and it has been proposed that other endonucleases may perform
  redundant functions in the absence of EXO1.) RPA binds the single-stranded region
  and a new strand is synthesized across the gap by DNA polymerase delta. The remaining
  nick is sealed by DNA ligase I (LIG1).<br>Concentrations of MMR proteins MSH2:MSH6
  and MLH1:PMS2 increase in human cells during S phase and are at their highest level
  and activity during this phase of the cell cycle (Edelbrock et al. 2009). Defects
  in MSH2, MSH6, MLH1, and PMS2 cause hereditary nonpolyposis colorectal cancer (HNPCC,
  also known as Lynch syndrome) (reviewed in Martin-Lopez and Fishel 2013).  View
  original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=5358508 Reactome].'
last-edited: 2021-01-25
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP3381
- /instance/WP3381
revision: null
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP3381.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: 'The mismatch repair (MMR) system corrects single base mismatches and
    small insertion and deletion loops (IDLs) of unpaired bases. MMR is primarily
    associated with DNA replication and is highly conserved across prokaryotes and
    eukaryotes. MMR consists of the following basic steps: a sensor (MutS homologue)
    detects a mismatch or IDL, the sensor activates a set of proteins (a MutL homologue
    and an exonuclease) that select the nascent DNA strand to be repaired, nick the
    strand, exonucleolytically remove  a region of nucleotides containing the mismatch,
    and finally a DNA polymerase resynthesizes the strand and a ligase seals the remaining
    nick (reviewed in Kolodner and Marsischkny 1999, Iyer et al. 2006, Li 2008, Fukui
    2010, Jiricny 2013). <br>Humans have 2 different MutS complexes. The MSH2:MSH6
    heterodimer (MutSalpha) recognizes single base mismatches and small loops of one
    or two unpaired bases. The MSH2:MSH3 heterodimer (MutSbeta) recognizes loops of
    two or more unpaired bases. Upon binding a mismatch, the MutS complex becomes
    activated in an ATP-dependent manner allowing for subsequent downstream interactions
    and movement on the DNA substrate. (There are two mechanisms proposed: a sliding
    clamp and a switch diffusion model.) Though the order of steps and structural
    details are not fully known, the activated MutS complex interacts with MLH1:PMS2
    (MutLalpha) and PCNA, the sliding clamp present at replication foci. The role
    of PCNA is multifaceted as it may act as a processivity factor in recruiting MMR
    proteins to replicating DNA, interact with MLH1:PMS2 and Exonuclease 1 (EXO1)
    to initiate excision of the recently replicated strand and direct DNA polymerase
    delta to initiate replacement of bases. MLH1:PMS2 makes an incision in the strand
    to be repaired and EXO1 extends the incision to make a single-stranded gap of
    up to 1 kb that removes the mismatched base(s). (Based on assays of purified human
    proteins, there is also a variant of the mismatch repair pathway that does not
    require EXO1, however the mechanism is not clear. EXO1 is almost always required,
    it is possible that the exonuclease activity of DNA polymerase delta may compensate
    in some situations and it has been proposed that other endonucleases may perform
    redundant functions in the absence of EXO1.) RPA binds the single-stranded region
    and a new strand is synthesized across the gap by DNA polymerase delta. The remaining
    nick is sealed by DNA ligase I (LIG1).<br>Concentrations of MMR proteins MSH2:MSH6
    and MLH1:PMS2 increase in human cells during S phase and are at their highest
    level and activity during this phase of the cell cycle (Edelbrock et al. 2009).
    Defects in MSH2, MSH6, MLH1, and PMS2 cause hereditary nonpolyposis colorectal
    cancer (HNPCC, also known as Lynch syndrome) (reviewed in Martin-Lopez and Fishel
    2013).  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=5358508
    Reactome].'
  keywords:
  - (5' phosphate, 3'
  - ADP
  - 'ADP '
  - AMP
  - ATP
  - 'ATP '
  - DNA Polymerase delta
  - DNA containing
  - 'DNA containing 1-2 base mismatch and single-strand incision '
  - DNA containing IDL
  - 'DNA containing IDL greater than 1 base and single strand incision '
  - 'DNA containing IDL of 2 or more bases '
  - 'DNA containing mismatch or 1-2 base IDL '
  - 'DNA containing single strand gap 150-1000 bp '
  - EXO1
  - 'EXO1 '
  - EXO1:MLH1:PMS2:MSH2:MSH3:ATP:PCNA:DNA containing IDL of 2 or more bases and single
    strand incision
  - EXO1:MLH1:PMS2:MSH2:MSH6:ATP:PCNA:DNA containing 1-2 base mismatch and single
    strand incision
  - LIG1
  - MLH1
  - 'MLH1 '
  - MLH1:PMS2
  - MLH1:PMS2:MSH2:MSH3:ATP:PCNA:DNA containing IDL of 2 bases or more
  - MLH1:PMS2:MSH2:MSH3:ATP:PCNA:DNA containing IDL of 2 or more bases and single
    strand incision
  - MLH1:PMS2:MSH2:MSH6:ATP:PCNA:DNA containing 1-2 base mismatch
  - MLH1:PMS2:MSH2:MSH6:ATP:PCNA:DNA containing 1-2 base mismatch and single strand
    incision
  - MSH2
  - 'MSH2 '
  - MSH2:MSH3:ADP
  - MSH2:MSH3:ADP:DNA
  - MSH2:MSH3:ATP:DNA
  - MSH2:MSH6:ADP
  - MSH2:MSH6:ADP:DNA
  - MSH2:MSH6:ATP:DNA
  - MSH3
  - 'MSH3 '
  - MSH6
  - 'MSH6 '
  - 'PCNA '
  - PCNA homotrimer
  - PMS2
  - 'PMS2 '
  - 'POLD1 '
  - 'POLD2 '
  - 'POLD3 '
  - 'POLD4 '
  - PPi
  - RPA heterotrimer
  - 'RPA1 '
  - 'RPA2 '
  - 'RPA3 '
  - RPA:DNA containing
  - Repaired
  - base IDL
  - bases or more
  - containing 1-2 base
  - containing IDL of 2
  - dATP
  - dCTP
  - dGTP
  - dTTP
  - double-stranded DNA
  - hydroxyl)
  - mismatch
  - mismatch or 1-2
  - of 2 or more bases
  - single strand gap
  - single strand nick
  - tetramer
  license: CC0
  name: Mismatch Repair
seo: CreativeWork
title: Mismatch Repair
wpid: WP3381
---