---
authors:
- ReactomeTeam
- Mkutmon
description: Human ribosomal RNAs (rRNAs) contain about 200 residues that are enzymatically
  modified after transcription in the nucleolus (Maden and Khan 1977, Maden 1988,
  Maden and Hughes 1997, reviewed in Hernandez-Verdun et al. 2010, Boschi-Muller and
  Motorin 2013). The modified residues occur in regions of the rRNAs that are located
  in functionally important parts of the ribosome, notably in the A and P peptidyl
  transfer sites, the polypeptide exit tunnel, and intersubunit contacts (Polikanov
  et al. 2015, reviewed in Decatur and Fournier 2002, Chow et al. 2007, Sharma and
  Lafontaine 2015). The two most common modifications are pseudouridines and 2'-O-methylribonucleotides.
  Formation of pseudouridine from encoded uridine is catalyzed by box H/ACA small
  nucleolar ribonucleoprotein (snoRNP) complexes (reviewed in Hamma and Ferre-D'Amare
  2010, Watkins and Bohnsack 2011, Ge and Yu 2013, Kierzek et al. 2014, Yu and Meier
  2014) and methylation of the hydroxyl group of the 2' carbon is catalyzed by box
  C/D snoRNPs (Kiss-Laszlo et al. 1996, Lapinaite et al. 2013, reviewed in Watkins
  and Bohnsack 2011). The snoRNP complexes contain common sets of protein subunits
  and unique snoRNAs that guide each complex to its target nucleotide of the rRNA
  by base-pairing between the snoRNA and the rRNA (reviewed in Henras et al. 2004,
  Watkins and Bohnsack 2011). Other modifications of rRNA include 5-methylcytidine
  (reviewed in Squires and Preiss 2010), 1-methylpseudouridine, 7-methylguanosine,
  6-dimethyladenosine, and 4-acetylcytidine (reviewed in Sharma and Lafontaine 2015).
  In yeast most modifications are introduced co-transcriptionally (Kos and Tollervey
  2010, reviewed in Turowski and Tollervey 2015), however the order of modification
  events and pre-rRNA cleavage events is not well characterized.  View original pathway
  at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=6790901 Reactome].
last-edited: 2021-01-25
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP3828
- /instance/WP3828
revision: null
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP3828.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: Human ribosomal RNAs (rRNAs) contain about 200 residues that are enzymatically
    modified after transcription in the nucleolus (Maden and Khan 1977, Maden 1988,
    Maden and Hughes 1997, reviewed in Hernandez-Verdun et al. 2010, Boschi-Muller
    and Motorin 2013). The modified residues occur in regions of the rRNAs that are
    located in functionally important parts of the ribosome, notably in the A and
    P peptidyl transfer sites, the polypeptide exit tunnel, and intersubunit contacts
    (Polikanov et al. 2015, reviewed in Decatur and Fournier 2002, Chow et al. 2007,
    Sharma and Lafontaine 2015). The two most common modifications are pseudouridines
    and 2'-O-methylribonucleotides. Formation of pseudouridine from encoded uridine
    is catalyzed by box H/ACA small nucleolar ribonucleoprotein (snoRNP) complexes
    (reviewed in Hamma and Ferre-D'Amare 2010, Watkins and Bohnsack 2011, Ge and Yu
    2013, Kierzek et al. 2014, Yu and Meier 2014) and methylation of the hydroxyl
    group of the 2' carbon is catalyzed by box C/D snoRNPs (Kiss-Laszlo et al. 1996,
    Lapinaite et al. 2013, reviewed in Watkins and Bohnsack 2011). The snoRNP complexes
    contain common sets of protein subunits and unique snoRNAs that guide each complex
    to its target nucleotide of the rRNA by base-pairing between the snoRNA and the
    rRNA (reviewed in Henras et al. 2004, Watkins and Bohnsack 2011). Other modifications
    of rRNA include 5-methylcytidine (reviewed in Squires and Preiss 2010), 1-methylpseudouridine,
    7-methylguanosine, 6-dimethyladenosine, and 4-acetylcytidine (reviewed in Sharma
    and Lafontaine 2015). In yeast most modifications are introduced co-transcriptionally
    (Kos and Tollervey 2010, reviewed in Turowski and Tollervey 2015), however the
    order of modification events and pre-rRNA cleavage events is not well characterized.  View
    original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=6790901 Reactome].
  keywords:
  - 1-methyl-3-aminocarboxypropylpseudoU-1248
  - 1-methylpseudoU-1248
  - 18SE
  - 2'-O-methylribonucleotides
  - 21S
  - 28S rRNA containing
  - 4-acetylcytidine-1377,1842
  - 5-methylC-4447
  - 6-dimethylA-1850,1851
  - 7-methylG-1639
  - 7-methylG-1639,
  - ADP
  - ATP
  - Ac-CoA
  - AdoHcy
  - AdoMet
  - 'BMS1 '
  - 'Box C/D snoRNA '
  - Box C/D snoRNP
  - 'Box H/ACA snoRNA '
  - Box H/ACA snoRNP
  - 'CIRH1A '
  - CoA-SH
  - 'DCAF13 '
  - 'DDX47 '
  - 'DDX49 '
  - 'DDX52 '
  - 'DHX37 '
  - 'DIEXF '
  - DIMT1
  - 'DKC1 '
  - 'EMG1 '
  - 'FBL '
  - 'FCF1 '
  - 'GAR1 '
  - 'HEATR1 '
  - 'IMP3 '
  - 'IMP4 '
  - 'KRR1 '
  - 'MPHOSPH10 '
  - MTAD
  - 'NAT10 '
  - NAT10:THUMPD1
  - 'NHP2 '
  - 'NHP2L1 '
  - 'NOC4L '
  - 'NOL11 '
  - 'NOL6 '
  - 'NOP10 '
  - 'NOP14 '
  - NOP2
  - 'NOP56 '
  - 'NOP58 '
  - 'PDCD11 '
  - PNO1
  - 'PWP2 '
  - Pi
  - 'RCL1 '
  - 'RPS14 '
  - 'RPS2 '
  - 'RPS6 '
  - 'RPS7 '
  - 'RPS9 '
  - 'RRP36 '
  - 'RRP7A '
  - 'RRP9 '
  - SSU Processome
  - 'TBL3 '
  - 'THUMPD1 '
  - 'TRMT112 '
  - TSR3
  - 'U3 snoRNA '
  - 'UTP11L '
  - 'UTP14A '
  - 'UTP14C '
  - 'UTP15 '
  - 'UTP18 '
  - 'UTP20 '
  - 'UTP3 '
  - 'UTP6 '
  - 'WBSCR22 '
  - WBSCR22:TRIMT112
  - 'WDR3 '
  - 'WDR36 '
  - 'WDR43 '
  - 'WDR46 '
  - 'WDR75 '
  - containing
  - nascent pre-rRNA
  - pre-rRNA
  - pre-rRNA containing
  - pseudouridine
  - transcript
  license: CC0
  name: rRNA modification in the nucleus and cytosol
seo: CreativeWork
title: rRNA modification in the nucleus and cytosol
wpid: WP3828
---