---
authors:
- ReactomeTeam
description: 'Ubiquitin is a small, 76 amino acid residue protein that is conjugated
  by E3 ubiquitin ligases to other proteins in order to regulate their function or
  degradation (enzymatic cascade reviewed in Neutzner and Neutzner 2012, Kleiger and
  Mayor 2014, structures and mechanisms of conjugating enzymes reviewed in Lorenz
  et al. 2013). Ubiquitination of target proteins usually occurs between the C-terminal
  glycine residue of ubiquitin and a lysine residue of the target, although linkages
  with cysteine, serine, and threonine residues are also observed (reviewed in Wang
  et al. 2012, McDowell and Philpott 2013).<br>Ubiquitin must first be processed from
  larger precursors and then activated by formation of a thiol ester bond between
  ubiquitin and an E1 activating enzyme (UBA1 or UBA6) and transfer to an E2 conjugating
  enzyme before being transferred by an E3 ligase to a target protein. Precursor proteins
  containing multiple ubiquitin monomers (polyubiquitins) are produced from the UBB
  and UBC genes; precursors containing a single ubiquitin monomer and a ribosomal
  protein are produced from the UBA52 and RPS27A genes. Many proteases (deubiquitinases)
  may potentially process these precursors yielding monomeric ubiquitin. The proteases
  OTULIN and USP5 are particularly active in cleaving the polyubiquitin precursors,
  whereas the proteases UCHL3, USP7, and USP9X cleave the ubiquitin-ribosomal protein
  precursors yielding ubiquitin monomers (Grou et al. 2015). A resultant ubiquitin
  monomer is activated by adenylation of the C-terminal glycine followed by conjugation
  of the C-terminus to a cysteine residue of the E1 enzymes UBA1 or UBA6 via a thiol
  ester bond. The ubiquitin is then transferred from the E1 enzyme to a cysteine residue
  of one of several E2 enzymes (reviewed in van Wijk and Timmers 2010, Stewart et
  al. 2016). Through a less well characterized mechanism, E3 ubiquitin ligases then
  bring a target protein and the E2-ubiquitin conjugate into proximity so that the
  ubiquitin is transferred via formation of an amide bond to a particular lysine residue
  (or, in rarer cases, a thiol ester bond to a cysteine residue or an ester bond to
  a serine or threonine residue) of the target protein (reviewed in Berndsen and Wolberger
  2014). Based on protein homologies, families of E3 ubiquitin ligases have been identified
  that include RING-type ligases (reviewed in Deshaies et al. 2009, Metzger et al.
  2012, Metzger et al. 2014), HECT-type ligases (reviewed in Rotin et al. 2009, Metzger
  et al. 2012), and RBR-type ligases (reviewed in Dove et al. 2016). A subset of the
  RING-type ligases participate in CULLIN-RING ligase complexes (CRLs which include
  SCF complexes, reviewed in Lee and Zhou 2007, Genschik et al. 2013, Skaar et al.
  2013, Lee et al. 2014).<br>Some E3-E2 combinations catalyze mono-ubiquitination
  of the target protein (reviewed in Nakagawa and Nakayama 2015). Other E3-E2 combinations
  catalyze conjugation of further ubiquitin monomers to the initial ubiquitin, forming
  polyubiquitin chains. (It may also be possible for some E3-E2 combinations to preassemble
  polyubiquitin and transfer it as a unit to the target protein.) Ubiquitin contains
  several lysine (K) residues and a free alpha amino group  to which further ubiquitin
  can be conjugated. Thus different types of polyubiquitin are possible: K11 linked
  polyubiquitin is observed in endoplasmic reticulum-associated degradation (ERAD),
  K29 linked polyubiquitin is observed in lysosomal degradation, K48 linked polyubiquitin
  directs target proteins to the proteasome for degradation, whereas K63 linked polyubiquitin
  generally acts as a scaffold to recruit other proteins in several cellular processes,
  notably DNA repair (reviewed in Komander et al. 2009). Ubiquitination is highly
  regulated (reviewed in Vittal et al. 2015) and affects all cellular processes including
  DNA damage response (reviewed in Brown and Jackson 2015), immune signaling (reviewed
  in Park et al. 2014, Lutz-Nicoladoni et al. 2015), and regulation of normal and
  cancerous cell growth (reviewed in Skaar and Pagano 2009, Yerlikaya and Yontem 2013,
  Strikoudis et al. 2014).  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=8852135
  Reactome].'
last-edited: 2021-01-25
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP4125
- /instance/WP4125
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP4125.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: 'Ubiquitin is a small, 76 amino acid residue protein that is conjugated
    by E3 ubiquitin ligases to other proteins in order to regulate their function
    or degradation (enzymatic cascade reviewed in Neutzner and Neutzner 2012, Kleiger
    and Mayor 2014, structures and mechanisms of conjugating enzymes reviewed in Lorenz
    et al. 2013). Ubiquitination of target proteins usually occurs between the C-terminal
    glycine residue of ubiquitin and a lysine residue of the target, although linkages
    with cysteine, serine, and threonine residues are also observed (reviewed in Wang
    et al. 2012, McDowell and Philpott 2013).<br>Ubiquitin must first be processed
    from larger precursors and then activated by formation of a thiol ester bond between
    ubiquitin and an E1 activating enzyme (UBA1 or UBA6) and transfer to an E2 conjugating
    enzyme before being transferred by an E3 ligase to a target protein. Precursor
    proteins containing multiple ubiquitin monomers (polyubiquitins) are produced
    from the UBB and UBC genes; precursors containing a single ubiquitin monomer and
    a ribosomal protein are produced from the UBA52 and RPS27A genes. Many proteases
    (deubiquitinases) may potentially process these precursors yielding monomeric
    ubiquitin. The proteases OTULIN and USP5 are particularly active in cleaving the
    polyubiquitin precursors, whereas the proteases UCHL3, USP7, and USP9X cleave
    the ubiquitin-ribosomal protein precursors yielding ubiquitin monomers (Grou et
    al. 2015). A resultant ubiquitin monomer is activated by adenylation of the C-terminal
    glycine followed by conjugation of the C-terminus to a cysteine residue of the
    E1 enzymes UBA1 or UBA6 via a thiol ester bond. The ubiquitin is then transferred
    from the E1 enzyme to a cysteine residue of one of several E2 enzymes (reviewed
    in van Wijk and Timmers 2010, Stewart et al. 2016). Through a less well characterized
    mechanism, E3 ubiquitin ligases then bring a target protein and the E2-ubiquitin
    conjugate into proximity so that the ubiquitin is transferred via formation of
    an amide bond to a particular lysine residue (or, in rarer cases, a thiol ester
    bond to a cysteine residue or an ester bond to a serine or threonine residue)
    of the target protein (reviewed in Berndsen and Wolberger 2014). Based on protein
    homologies, families of E3 ubiquitin ligases have been identified that include
    RING-type ligases (reviewed in Deshaies et al. 2009, Metzger et al. 2012, Metzger
    et al. 2014), HECT-type ligases (reviewed in Rotin et al. 2009, Metzger et al.
    2012), and RBR-type ligases (reviewed in Dove et al. 2016). A subset of the RING-type
    ligases participate in CULLIN-RING ligase complexes (CRLs which include SCF complexes,
    reviewed in Lee and Zhou 2007, Genschik et al. 2013, Skaar et al. 2013, Lee et
    al. 2014).<br>Some E3-E2 combinations catalyze mono-ubiquitination of the target
    protein (reviewed in Nakagawa and Nakayama 2015). Other E3-E2 combinations catalyze
    conjugation of further ubiquitin monomers to the initial ubiquitin, forming polyubiquitin
    chains. (It may also be possible for some E3-E2 combinations to preassemble polyubiquitin
    and transfer it as a unit to the target protein.) Ubiquitin contains several lysine
    (K) residues and a free alpha amino group  to which further ubiquitin can be conjugated.
    Thus different types of polyubiquitin are possible: K11 linked polyubiquitin is
    observed in endoplasmic reticulum-associated degradation (ERAD), K29 linked polyubiquitin
    is observed in lysosomal degradation, K48 linked polyubiquitin directs target
    proteins to the proteasome for degradation, whereas K63 linked polyubiquitin generally
    acts as a scaffold to recruit other proteins in several cellular processes, notably
    DNA repair (reviewed in Komander et al. 2009). Ubiquitination is highly regulated
    (reviewed in Vittal et al. 2015) and affects all cellular processes including
    DNA damage response (reviewed in Brown and Jackson 2015), immune signaling (reviewed
    in Park et al. 2014, Lutz-Nicoladoni et al. 2015), and regulation of normal and
    cancerous cell growth (reviewed in Skaar and Pagano 2009, Yerlikaya and Yontem
    2013, Strikoudis et al. 2014).  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=8852135
    Reactome].'
  keywords:
  - ATP
  - RPS27A
  - 'UBA52(1-76) '
  - 'Ub-C95-UBE2S '
  - adenylate
  - 'UCHL3 '
  - 'UBE2S '
  - E3 ubiquitin ligases
  - 'adenylatedG152-UBB(77-152) '
  - Ub
  - 'Ub-C85-UBE2D2 '
  - UBA1:ubiquitin
  - 'Ub-C88-UBE2B '
  - OTULIN,USP5
  - 'Ub-C131-UBE2E1 '
  - Ub:E2 (UBA1
  - UBA52(1-76)
  - conjugating enzyme
  - UBA52(1-128)
  - 'adenylatedG152-UBC(77-152) '
  - 'adenylatedG228-UBC(153-228) '
  - 'UBC(229-304) '
  - UBC(305-380)
  - 'adenylatedG304-UBC(229-304) '
  - 'Ub-C93-CDC34 '
  - UBA6:ubiquitin
  - UBC(153-228)
  - Ub-Cys632-UBA1:ubiquitin adenylate
  - 'Ub-C188-UBE2Z '
  - Ub:E2 (UBA6
  - Ub-Cys632-UBA1
  - UCHL3,USP7,USP9X
  - 'UBE2D2 '
  - 'adenylatedlG76-RPS27A(1-76) '
  - RPS27A(1-76)
  - 'Ub-C304-UBE2Q2 '
  - UBC(609-684)
  - 'Ub-C86-UBE2T '
  - 'UBC(381-456) '
  - 'UBC(305-380) '
  - 'Ub-C92-UBE2K '
  - 'USP7 '
  - 'adenylatedG684-UBC(609-684) '
  - RPS27A(77-156)
  - 'Ub-C93-UBE2R2 '
  - 'UBE2K '
  - 'Ub-C114-UBE2C '
  - E2 ubiquitin
  - 'UBA1 '
  - 'RPS27A(1-76) '
  - UBA6
  - 'adenylatedG76-UBC(1-76) '
  - 'UBE2B '
  - Ub-Cys625-UBA6
  - 'UBE2Q2 '
  - 'UBE2E1 '
  - proteins
  - 'adenylatedG228-UBB(153-228) '
  - 'UBC(533-608) '
  - 'Ub-C90-UBE2G1 '
  - Ub-Cys625-UBA6:ubiquitin adenylate
  - UBC(457-532)
  - 'adenylatedG76-UBA52(1-76) '
  - 'UBC(1-76) '
  - 'OTULIN '
  - 'UBE2W '
  - UBA1
  - 'USP5 '
  - 'Ub-C145-UBE2E3 '
  - 'UBC(153-228) '
  - 'adenylatedG76-UBB(1-76) '
  - UBC(77-152)
  - UBC(533-608)
  - 'adenylatedG380-UBC(305-380) '
  - RPL40
  - 'UBA6 '
  - UBC
  - substrate)
  - PPi
  - (UBA1 substrate)
  - 'UBC(77-152) '
  - 'Ub-C89-UBE2G2 '
  - UBC(381-456)
  - 'Ub-C91-UBE2W '
  - UBB(1-76)
  - UBC(229-304)
  - 'UBE2Z '
  - 'UBE2L3 '
  - UBB
  - 'UBC(457-532) '
  - UBB(77-152)
  - 'UBB(1-76) '
  - 'UBE2C '
  - 'UBE2G2 '
  - 'UBE2D1 '
  - ubiquitinate target
  - 'Ub-C87-UBE2H '
  - 'Ub-C85-UBE2D1 '
  - UBC(1-76)
  - 'UBB(153-228) '
  - 'USP9X '
  - UBB(153-228)
  - 'adenylatedG532-UBC(457-532) '
  - 'UBE2R2 '
  - 'UBE2T '
  - (UBA6 substrate)
  - 'Ub-C86-UBE2L3 '
  - 'UBC(609-684) '
  - AMP
  - 'CDC34 '
  - 'UBE2E3 '
  - 'adenylatedG608-UBC(533-608) '
  - 'UBE2G1 '
  - 'UBB(77-152) '
  - 'Ub-C88-UBE2A '
  - 'UBE2H '
  - 'adenylatedG456-UBC(381-456) '
  - 'UBE2A '
  license: CC0
  name: Protein ubiquitination
seo: CreativeWork
title: Protein ubiquitination
wpid: WP4125
---