---
annotations:
- type: Pathway Ontology
  value: signaling pathway
- type: Pathway Ontology
  value: signaling pathway in the innate immune response
authors:
- ReactomeTeam
- Egonw
- Anwesha
- Ryanmiller
description: The innate immune system is the first line of defense against invading
  microorganisms, a broad specificity response characterized by the recruitment and
  activation of phagocytes and the release of anti-bacterial peptides. The receptors
  involved recognize conserved molecules present in microbes called pathogen-associated
  molecular patterns (PAMPs), and/or molecules that are produced as a result of tissue
  injury, the damage associated molecular pattern molecules (DAMPs). PAMPs are essential
  to the pathogen and therefore unlikely to vary. Examples are lipopolysaccharide
  (LPS), peptidoglycans (PGNs) and viral RNA. DAMPs include intracellular proteins,
  such as heat-shock proteins and extracellular matrix proteins released by tissue
  injury, such as hyaluronan fragments. Non-protein DAMPs include ATP, uric acid,
  heparin sulfate and dsDNA. The receptors for these factors are referred to collectively
  as pathogen- or pattern-recognition receptors (PRRs). The best studied of these
  are the membrane-associated Toll-like receptor family. Less well studied but more
  numerous are the intracellular nucleotide-binding domain, leucine rich repeat containing
  receptors (NLRs) also called nucleotide binding oligomerization domain (NOD)-like
  receptors, a family with over 20 members in humans and over 30 in mice. These recognise
  PAMPs/DAMPs from phagocytosed microorganisms or from intracellular infections (Kobayashi
  et al. 2003, Proell et al. 2008, Wilmanski et al. 2008). Some NLRs are involved
  in process unrelated to pathogen detection such as tissue homeostasis, apoptosis,
  graft-versus-host disease and early development (Kufer & Sansonetti 2011). <br><br><br>Structurally
  NLRs can be subdivided into the caspase-recruitment domain (CARD)-containing NLRCs
  (NODs) and the pyrin domain (PYD)-containing NLRPs (NALPs), plus outliers including
  ice protease (caspase-1) activating factor (IPAF) (Martinon & Tschopp, 2005). In
  practical terms, NLRs can be divided into the relatively well characterized NOD1/2
  which signal via RIP2 primarily to NFkappaB, and the remainder, some of which participate
  in macromolecular structures called Inflammasomes that activate caspases. Mutations
  in several members of the NLR protein family have been linked to inflammatory diseases,
  suggesting these molecules play important roles in maintaining host-pathogen interactions
  and inflammatory responses.<br><br><br>Most NLRs have a tripartite structure consisting
  of a variable amino-terminal domain, a central nucleotide-binding oligomerization
  domain (NOD or NACHT) that is believed to mediate the formation of self oligomers,
  and a carboxy-terminal leucine-rich repeat (LRR) that detects PAMPs/DAMPs. In most
  cases the amino-terminal domain includes protein-interaction modules, such as CARD
  or PYD, some harbour baculovirus inhibitor repeat (BIR) or other domains. For most
  characterised NLRs these domains have been attributed to downstream signaling<br><br>Under
  resting conditions, NLRs are thought to be present  in an autorepressed form, with
  the LRR folded back onto the NACHT domain preventing oligomerization. Accessory
  proteins may help maintain the inactive state. PAMP/DAMP exposure is thought to
  triggers conformational changes that expose the NACHT domain enabling oligomerization
  and recruitment of effectors, though it should be noted that due to the lack of
  availability of structural data, the mechanistic details of NLR activation remain
  largely elusive.<br><br>New terminology for NOD-like receptors was adopted by the
  Human Genome Organization (HUGO) in 2008 to standardize the nomenclature of NLRs.
  The acronym NLR, once standing for NOD-like receptor, now is an abbreviation of
  'nucleotide-binding domain, leucine-rich repeat containing' protein. The term NOD-like
  receptor is officially outdated and replaced by NLRC where the C refers to the CARD
  domain. However the official gene symbols for NOD1 and NOD2 still contain NOD and
  this general term is still widely used.  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=168643
  Reactome].
last-edited: 2021-01-25
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP2763
- /instance/WP2763
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP2763.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: The innate immune system is the first line of defense against invading
    microorganisms, a broad specificity response characterized by the recruitment
    and activation of phagocytes and the release of anti-bacterial peptides. The receptors
    involved recognize conserved molecules present in microbes called pathogen-associated
    molecular patterns (PAMPs), and/or molecules that are produced as a result of
    tissue injury, the damage associated molecular pattern molecules (DAMPs). PAMPs
    are essential to the pathogen and therefore unlikely to vary. Examples are lipopolysaccharide
    (LPS), peptidoglycans (PGNs) and viral RNA. DAMPs include intracellular proteins,
    such as heat-shock proteins and extracellular matrix proteins released by tissue
    injury, such as hyaluronan fragments. Non-protein DAMPs include ATP, uric acid,
    heparin sulfate and dsDNA. The receptors for these factors are referred to collectively
    as pathogen- or pattern-recognition receptors (PRRs). The best studied of these
    are the membrane-associated Toll-like receptor family. Less well studied but more
    numerous are the intracellular nucleotide-binding domain, leucine rich repeat
    containing receptors (NLRs) also called nucleotide binding oligomerization domain
    (NOD)-like receptors, a family with over 20 members in humans and over 30 in mice.
    These recognise PAMPs/DAMPs from phagocytosed microorganisms or from intracellular
    infections (Kobayashi et al. 2003, Proell et al. 2008, Wilmanski et al. 2008).
    Some NLRs are involved in process unrelated to pathogen detection such as tissue
    homeostasis, apoptosis, graft-versus-host disease and early development (Kufer
    & Sansonetti 2011). <br><br><br>Structurally NLRs can be subdivided into the caspase-recruitment
    domain (CARD)-containing NLRCs (NODs) and the pyrin domain (PYD)-containing NLRPs
    (NALPs), plus outliers including ice protease (caspase-1) activating factor (IPAF)
    (Martinon & Tschopp, 2005). In practical terms, NLRs can be divided into the relatively
    well characterized NOD1/2 which signal via RIP2 primarily to NFkappaB, and the
    remainder, some of which participate in macromolecular structures called Inflammasomes
    that activate caspases. Mutations in several members of the NLR protein family
    have been linked to inflammatory diseases, suggesting these molecules play important
    roles in maintaining host-pathogen interactions and inflammatory responses.<br><br><br>Most
    NLRs have a tripartite structure consisting of a variable amino-terminal domain,
    a central nucleotide-binding oligomerization domain (NOD or NACHT) that is believed
    to mediate the formation of self oligomers, and a carboxy-terminal leucine-rich
    repeat (LRR) that detects PAMPs/DAMPs. In most cases the amino-terminal domain
    includes protein-interaction modules, such as CARD or PYD, some harbour baculovirus
    inhibitor repeat (BIR) or other domains. For most characterised NLRs these domains
    have been attributed to downstream signaling<br><br>Under resting conditions,
    NLRs are thought to be present  in an autorepressed form, with the LRR folded
    back onto the NACHT domain preventing oligomerization. Accessory proteins may
    help maintain the inactive state. PAMP/DAMP exposure is thought to triggers conformational
    changes that expose the NACHT domain enabling oligomerization and recruitment
    of effectors, though it should be noted that due to the lack of availability of
    structural data, the mechanistic details of NLR activation remain largely elusive.<br><br>New
    terminology for NOD-like receptors was adopted by the Human Genome Organization
    (HUGO) in 2008 to standardize the nomenclature of NLRs. The acronym NLR, once
    standing for NOD-like receptor, now is an abbreviation of 'nucleotide-binding
    domain, leucine-rich repeat containing' protein. The term NOD-like receptor is
    officially outdated and replaced by NLRC where the C refers to the CARD domain.
    However the official gene symbols for NOD1 and NOD2 still contain NOD and this
    general term is still widely used.  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=168643
    Reactome].
  keywords:
  - 'MAPK14 '
  - 'BCL2L1 '
  - CYLD
  - 'K63polyUb-TRAF6 '
  - PSTPIP1 trimer
  - HSP90AB1
  - P2RX7
  - 'K63polyUb '
  - 'PSTPIP1 '
  - 'BIRC3 '
  - NLRP3 elicitor
  - Oxidized
  - NOD1:iE-DAP
  - 'Double-stranded DNA '
  - 'iE-DAP '
  - 'NLRP3 '
  - 'MEFV '
  - ligases
  - 'CHUK '
  - 'MAPK11 '
  - 'p-2S,S376,T,T209,T387-IRAK1 '
  - AAMP
  - 'TAB2 '
  - Long prodomain
  - TAK1 complex
  - AAMP:NOD2
  - ubiquitin ligase
  - 'NOD1 '
  - ATP:P2X7
  - TXNIP
  - Pyrin trimer:ASC
  - proteins
  - ROS
  - oligomer:K63-polyUb-RIP2:NEMO:TAK1 complex
  - 'NFKB2(1-454) '
  - proteins:NLRP3
  - 'TXN '
  - 'MAPK13 '
  - PAMP:NOD
  - TXN
  - 'PYCARD '
  - Phospho-p38 MAPK
  - prodomain caspases
  - AIM2
  - Double-stranded DNA
  - TXNIP:NLRP3
  - 'TAB1 '
  - 'p-IRAK2 '
  - SUGT1:HSP90
  - 'NOD2 '
  - 'p-T180,Y182-MAPK13 '
  - PANX1
  - 'NFKB1(1-433) '
  - 'MAPK12 '
  - 'UBE2N '
  - IPAF elicitors
  - CARD9
  - IPAF elicitors:NLRC4
  - RIP2 ubiquitin
  - IPAF
  - '2xHC-TXN '
  - 2xHC-TXN
  - 'MDP '
  - 'BCL2 '
  - 'NLRC4 '
  - TRAF6 E3/E2
  - RIPK2
  - Pyrin trimer
  - 'CASP2(2-452) '
  - ATP:P2X7 oligomer
  - oligomer:RIP2:CARD9
  - 'ITCH '
  - PAMP:NOD oligomer
  - oligomer:ASC:Procaspase-1
  - 'CASP1(1-404) '
  - thioredoxin:TXNIP
  - 'IKBKB '
  - 'Ub-209-RIPK2 '
  - 'CASP8(1-479) '
  - K63polyUb
  - elicitors:NLRP3
  - 'NLRP1 '
  - dsDNA:AIM2
  - MDP:NLRP1:ATP
  - PYCARD
  - complexes
  - 'CARD9 '
  - 'P2RX7 '
  - 'TAB3 '
  - 'p-S177,S181-IKBKB '
  - 'SiO2 '
  - MDP:NLRP1
  - 'UBE2V1 '
  - TNFAIP3
  - Thioredoxin:TXNIP
  - ATP
  - trimer
  - 'BIRC2 '
  - molecules:NLRP3
  - 'p-T183,Y185-MAPK12 '
  - 'ATP '
  - MAP2K6
  - oligomer:Pannexin-1
  - 'Flagellin '
  - oligomer:RIP2:K63-pUb-K285-NEMO
  - 'prgJ '
  - Bcl-2/Bcl-X(L)
  - 'p-T180,Y182-MAPK11 '
  - NOD1:iE-DAP:Long
  - NLRP3 elicitor small
  - NOD1
  - 'CASP4(?-377) '
  - 'p-T180,Y182-MAPK14 '
  - oligomer:K63-polyUb-RIP2:NEMO
  - NLRP3 gene
  - p-S207,T211-MAP2K6
  - 'IKBKG '
  - oligomer:RIP2
  - NLRP3:SUGT1:HSP90
  - NOD2
  - molecules
  - 'RIPK2 '
  - oligomer:K63-polyUb-RIP2:NEMO:activated TAK1 complex
  - NLRC4
  - 'APP(672-711) '
  - ADP
  - 'dsDNA:AIM2 oligomer '
  - SUGT1
  - IKBKG:p-S176,S180-CHUK:p-S177,S181-IKBKB
  - CASP1(1-404)
  - 'HSP90AB1 '
  - 'Asb '
  - oligomer
  - PSTPIP1 trimer:Pyrin
  - 'AIM2 '
  - 'Alpha-hemolysin '
  - MDP
  - NLRP3
  - K+
  - elicitors:NLRC4:Procaspase-1
  - dsDNA:AIM2 oligomer
  - IKBKG
  - CHUK:IKBKB:IKBKG
  - 'CASP9(1-416) '
  - 'MAP3K7 '
  - MDP:NOD2 oligomer
  - 'TRAF6 '
  - 'SUGT1 '
  - caspases
  - 'HUA '
  - 'p-T184,T187-MAP3K7 '
  - 'p-S176,S180-CHUK '
  - 'TXNIP '
  - complex
  - Bcl-2/Bcl-X(L):NLRP1
  - NFkB Complex
  - 'RELA '
  - NOD1:iE-DAP oligomer
  - NLRP1
  - Activated TAK
  - MDP:NOD2
  - iE-DAP
  - 'PANX1 '
  - 'Ub-285-IKBKG '
  - 'NLRP3 elicitors:NLRP3 oligomer '
  - oligomer:RIP2:NEMO
  - p38 MAPK
  - 'AAMP '
  - oligomer:ASC
  license: CC0
  name: Nucleotide-binding domain, leucine rich repeat containing receptor (NLR) signaling
    pathways
seo: CreativeWork
title: Nucleotide-binding domain, leucine rich repeat containing receptor (NLR) signaling
  pathways
wpid: WP2763
---