---
annotations:
- id: PW:0000003
  parent: signaling pathway
  type: Pathway Ontology
  value: signaling pathway
authors:
- ReactomeTeam
- Egonw
- DeSl
description: The 22 members of the fibroblast growth factor (FGF) family of growth
  factors mediate their cellular responses by binding to and activating the different
  isoforms encoded by the four receptor tyrosine kinases (RTKs) designated FGFR1,
  FGFR2, FGFR3 and FGFR4. These receptors are key regulators of several developmental
  processes in which cell fate and differentiation to various tissue lineages are
  determined. Unlike other growth factors, FGFs act in concert with heparin or heparan
  sulfate proteoglycan (HSPG) to activate FGFRs and to induce the pleiotropic responses
  that lead to the variety of cellular responses induced by this large family of growth
  factors. An alternative, FGF-independent, source of FGFR activation originates from
  the interaction with cell adhesion molecules, typically in the context of interactions
  on neural cell membranes and is crucial for neuronal survival and development.<br><br>Upon
  ligand binding, receptor dimers are formed and their intrinsic tyrosine kinase is
  activated causing phosphorylation of multiple tyrosine residues on the receptors.
  These then serve as docking sites for the recruitment of SH2 (src homology-2) or
  PTB (phosphotyrosine binding) domains of adaptors, docking proteins or signaling
  enzymes. Signaling complexes are assembled and recruited to the active receptors
  resulting in a cascade of phosphorylation events.<br><br>This leads to stimulation
  of intracellular signaling pathways that control cell proliferation, cell differentiation,
  cell migration, cell survival and cell shape, depending on the cell type or stage
  of maturation.<br>  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=5654738
  Reactome].
last-edited: 2020-10-30
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP3338
- /instance/WP3338
revision: null
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP3338.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: The 22 members of the fibroblast growth factor (FGF) family of growth
    factors mediate their cellular responses by binding to and activating the different
    isoforms encoded by the four receptor tyrosine kinases (RTKs) designated FGFR1,
    FGFR2, FGFR3 and FGFR4. These receptors are key regulators of several developmental
    processes in which cell fate and differentiation to various tissue lineages are
    determined. Unlike other growth factors, FGFs act in concert with heparin or heparan
    sulfate proteoglycan (HSPG) to activate FGFRs and to induce the pleiotropic responses
    that lead to the variety of cellular responses induced by this large family of
    growth factors. An alternative, FGF-independent, source of FGFR activation originates
    from the interaction with cell adhesion molecules, typically in the context of
    interactions on neural cell membranes and is crucial for neuronal survival and
    development.<br><br>Upon ligand binding, receptor dimers are formed and their
    intrinsic tyrosine kinase is activated causing phosphorylation of multiple tyrosine
    residues on the receptors. These then serve as docking sites for the recruitment
    of SH2 (src homology-2) or PTB (phosphotyrosine binding) domains of adaptors,
    docking proteins or signaling enzymes. Signaling complexes are assembled and recruited
    to the active receptors resulting in a cascade of phosphorylation events.<br><br>This
    leads to stimulation of intracellular signaling pathways that control cell proliferation,
    cell differentiation, cell migration, cell survival and cell shape, depending
    on the cell type or stage of maturation.<br>  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=5654738
    Reactome].
  keywords:
  - (A:C):S112/S115
  - ADP
  - ATP
  - 'AZ 2171 '
  - 'AZD4547 '
  - Activated
  - Activated FGFR2
  - 'Activated FGFR2 ligand-independent mutants '
  - 'Activated FGFR2 mutants with enhanced kinase activity '
  - Activated FGFR2:FRS2
  - Activated FGFR2:FRS3
  - Activated FGFR2:SHC1
  - Activated FGFR2b
  - 'Activated FGFR2b homodimer bound to FGF '
  - Activated FGFR2c
  - 'Activated FGFR2c homodimer bound to FGF '
  - 'Activated overexpressed FGFR2 dimers '
  - BRAF
  - 'BRAF '
  - CBL
  - 'CBL '
  - DAG and IP3
  - ESRP1
  - 'ESRP1 '
  - ESRP2
  - 'ESRP2 '
  - FGF
  - 'FGF1 '
  - FGF1,2
  - 'FGF10 '
  - 'FGF16 '
  - 'FGF17-1 '
  - 'FGF18 '
  - 'FGF2(10-155) '
  - FGF2,7,10,22
  - 'FGF20 '
  - 'FGF22 '
  - 'FGF3 '
  - 'FGF4 '
  - 'FGF5-1 '
  - 'FGF6 '
  - 'FGF7 '
  - 'FGF8-1 '
  - 'FGF9 '
  - FGFBP
  - 'FGFBP1 '
  - 'FGFBP2 '
  - 'FGFBP3 '
  - FGFBP:FGF
  - FGFR2
  - 'FGFR2 '
  - FGFR2 IIIa TM
  - 'FGFR2 IIIa TM '
  - FGFR2 IIIb-specific
  - FGFR2 IIIc-specific
  - 'FGFR2 K660E '
  - 'FGFR2 K660M '
  - 'FGFR2 K660N '
  - 'FGFR2 L764fs*4 '
  - 'FGFR2 N549H '
  - 'FGFR2 N549K '
  - 'FGFR2 S267P '
  - 'FGFR2 W290C '
  - FGFR2 fusion dimers
  - FGFR2 fusions
  - 'FGFR2 ligand-independent mutant dimers '
  - FGFR2 mutant dimers
  - 'FGFR2 mutant dimers with enhanced kinase activity '
  - FGFR2 mutants with
  - FGFR2 point mutant
  - 'FGFR2(2-822)-CIT(927-2027) fusion '
  - 'FGFR2(22-767)-AFF3(292-1226) fusion '
  - 'FGFR2(22-767)-AHCYL1(108-530) fusion '
  - 'FGFR2(22-767)-CASP7(1-303) fusion '
  - 'FGFR2(22-767)-CCAR(51-923) fusion '
  - 'FGFR2(22-767)-CCDC6(102-474) fusion '
  - 'FGFR2(22-767)-OFD1(38-1012) fusion '
  - 'FGFR2(22-768)-BICC1(80-974) fusion '
  - FGFR2:PLCG1
  - FGFR2:TKIs
  - FGFR2:p-4Y-PLCG1
  - FGFR2:p-8T-FRS2
  - FGFR2:p-FRS
  - FGFR2:p-FRS2
  - FGFR2:p-FRS2:GRB2:GAB1:PI3K
  - FGFR2:p-FRS2:GRB2:GAB1:PIK3R1
  - FGFR2:p-FRS2:p-PTPN11
  - FGFR2:p-FRS2:p-PTPN11:GRB2:GAB1:PI3K
  - FGFR2:p-FRS2:p-PTPN11:GRB2:GAB1:PIK3R1
  - FGFR2:p-FRS2:p-PTPN11:p-CBL:GRB2
  - FGFR2:p-FRS3
  - FGFR2:p-FRS:GRB2:SOS1
  - FGFR2:p-FRS:PTPN11
  - FGFR2:p-FRS:p-PTPN11
  - FGFR2:pY-SHC1
  - FGFR2:pY-SHC1:GRB2:SOS1
  - FGFR2IIIa
  - FGFR2b
  - 'FGFR2b '
  - 'FGFR2b C3 variant '
  - 'FGFR2b C382R '
  - 'FGFR2b P253R '
  - 'FGFR2b S252W '
  - 'FGFR2b S373C '
  - 'FGFR2b Y376C '
  - FGFR2b homodimer
  - 'FGFR2b long '
  - FGFR2b mature mRNA
  - FGFR2b mutant dimers
  - FGFR2b mutants with
  - 'FGFR2b short '
  - FGFR2b, FGFR2c
  - FGFR2b-binding FGFs
  - 'FGFR2b-binding FGFs '
  - FGFR2c
  - 'FGFR2c '
  - 'FGFR2c A314D '
  - 'FGFR2c A314S '
  - 'FGFR2c A315S '
  - 'FGFR2c A315T '
  - 'FGFR2c P253R '
  - 'FGFR2c S252W '
  - 'FGFR2c S372C '
  - 'FGFR2c W290G '
  - 'FGFR2c Y375C '
  - FGFR2c homodimer
  - 'FGFR2c long '
  - FGFR2c mature mRNA
  - FGFR2c mutant
  - FGFR2c mutant dimers
  - FGFR2c mutants with
  - 'FGFR2c short '
  - FGFR2c-binding FGFs
  - 'FGFR2c-binding FGFs '
  - FGFs:FP-1039
  - FP-1039
  - 'FP-1039 '
  - FRS2
  - 'FRS2 '
  - FRS3
  - 'FRS3 '
  - 'GAB1 '
  - GDP
  - 'GDP '
  - GP369
  - 'GP369 '
  - GRB2-1
  - 'GRB2-1 '
  - GRB2-1:SOS1
  - GRB2:GAB1
  - GRB2:GAB1:PIK3R1
  - 'GTF2F1 '
  - 'GTF2F2 '
  - GTP
  - 'GTP '
  - 'GalNAc-T178-FGF23(25-251) '
  - HNRNPA1
  - 'HNRNPA1 '
  - 'HNRNPF '
  - 'HNRNPH1 '
  - HNRNPM
  - 'HNRNPM '
  - HS
  - 'HS '
  - 'NCBP1 '
  - 'NCBP2 '
  - Overexpressed
  - Overexpressed FGFR2
  - PI(3,4,5)P3
  - 'PI(3,4,5)P3 '
  - PI(4,5)P2
  - PIK3CA
  - 'PIK3CA '
  - PIK3R1
  - 'PIK3R1 '
  - PIP3 activates AKT
  - PLCG1
  - 'PLCG1 '
  - 'POLR2A '
  - 'POLR2B '
  - 'POLR2C '
  - 'POLR2D '
  - 'POLR2E '
  - 'POLR2F '
  - 'POLR2G '
  - 'POLR2H '
  - 'POLR2I '
  - 'POLR2J '
  - 'POLR2K '
  - 'POLR2L '
  - PP2A (A:C)
  - PP2A(A:C):S112/S121-pSPRY2
  - PP2A(A:C):SPRY2
  - PP2A(A:C):Y55/Y227-pSPRY2
  - PPA2A
  - PPA2A (A:C):Y55/Y227
  - PPA2A(A:C):SPRY2
  - 'PPP2CA '
  - 'PPP2CB '
  - 'PPP2R1A '
  - PTBP1
  - 'PTBP1 '
  - PTPN11
  - 'PTPN11 '
  - 'Phosphorylated Fibroblast growth factor receptor 2b short '
  - Pi
  - RAF/MAP kinase
  - 'RBFOX2 '
  - 'RPS27A(1-76) '
  - 'S-Farn-Me KRAS4B '
  - 'S-Farn-Me PalmS NRAS '
  - 'S-Farn-Me-2xPalmS HRAS '
  - 'S-Farn-Me-PalmS KRAS4A '
  - S111/S120
  - 'SHC1-2 '
  - SHC1-2,SHC1-3
  - 'SHC1-3 '
  - 'SOS1 '
  - 'SPRY2 '
  - SPRY2:B-RAF
  - SRC-1
  - 'TIA1 '
  - TIA1/TIAL1
  - 'TIAL1 '
  - TM:FGF1,2:FGFR2b,
  - Tyrosine kinase
  - 'UBA52(1-76) '
  - 'UBB(1-76) '
  - 'UBB(153-228) '
  - 'UBB(77-152) '
  - 'UBC(1-76) '
  - 'UBC(153-228) '
  - 'UBC(229-304) '
  - 'UBC(305-380) '
  - 'UBC(381-456) '
  - 'UBC(457-532) '
  - 'UBC(533-608) '
  - 'UBC(609-684) '
  - 'UBC(77-152) '
  - Ub
  - Ub-(Y55/Y227)p-SPRY2
  - Ub-Activated FGFR2
  - Ub:Y55/Y227-pSPRY2:CBL
  - Y55/Y227-pSPRY2:CBL
  - activated
  - activity
  - binding
  - binding FGFs
  - bound to FGF
  - bound to FGFs
  - capped, methylated
  - 'capped, methylated FGFR2 nascent transcript '
  - cascade
  - complex
  - complex:Ub-p-FRS2
  - dimers
  - dimers:TKIs
  - enhanced
  - enhanced kinase
  - enhanced ligand
  - hnRNPH1:hnPNPF:RBFOX2
  - homodimer bound to
  - homodimers
  - homodimers:GP369
  - inhibitors of
  - inhibitors of FGFR2
  - kinase activity
  - ligand-binding
  - ligand-independent
  - mutant dimers
  - mutant-binding
  - mutant-binding FGFs
  - mutants
  - mutants with
  - mutants:PLCG1
  - mutants:p-4Y-PLCG1
  - overexpressed FGFR2
  - p-4Y-PLCG1
  - 'p-4Y-PLCG1 '
  - 'p-5Y-FRS3 '
  - 'p-6Y FGFR2 (22-767)-CCAR(51-923) fusion '
  - 'p-6Y FGFR2(2-822)-CIT(927-2027) fusion '
  - 'p-6Y FGFR2(22-767)-AFF3(292-1226) fusion '
  - 'p-6Y FGFR2(22-767)-AHCYL1(108-530) fusion '
  - 'p-6Y FGFR2(22-767)-CASP7(1-303) fusion '
  - 'p-6Y FGFR2(22-767)-CCDC6(102-474) fusion '
  - 'p-6Y FGFR2(22-767)-OFD1(38-1012) fusion '
  - 'p-6Y FGFR2(22-768)-BICC1(80-974) fusion '
  - 'p-6Y-FGFR2b C3 variant '
  - 'p-6Y-FRS2 '
  - 'p-8T-FRS2 '
  - 'p-8Y-FGFR2 '
  - 'p-8Y-FGFR2 K660E '
  - 'p-8Y-FGFR2 K660M '
  - 'p-8Y-FGFR2 K660N '
  - 'p-8Y-FGFR2 N549H '
  - 'p-8Y-FGFR2 N549K '
  - 'p-8Y-FGFR2 S267P '
  - 'p-8Y-FGFR2 W290C '
  - 'p-8Y-FGFR2-3 '
  - 'p-8Y-FGFR2-5 '
  - 'p-8Y-FGFR2b C382R '
  - 'p-8Y-FGFR2b P253R '
  - 'p-8Y-FGFR2b S252W '
  - 'p-8Y-FGFR2b S373C '
  - 'p-8Y-FGFR2b S376C '
  - 'p-8Y-FGFR2c A314D '
  - 'p-8Y-FGFR2c A314S '
  - 'p-8Y-FGFR2c A315S mutant '
  - 'p-8Y-FGFR2c A315T '
  - 'p-8Y-FGFR2c P253R '
  - 'p-8Y-FGFR2c S252W '
  - 'p-8Y-FGFR2c S372C '
  - 'p-8Y-FGFR2c W290G '
  - 'p-8Y-FGFR2c Y375C '
  - 'p-8Y-FGFR2c long '
  - p-S111,S120-SPRY2
  - 'p-S111,S120-SPRY2 '
  - 'p-S112,S115-SPRY2 '
  - 'p-S112,S121-SPRY2 '
  - p-SPRY2
  - p-SPRY2:B-RAF
  - p-SPRY2:GRB2
  - p-T,Y MAPK dimers
  - 'p-T185,Y187-MAPK1 '
  - 'p-T202,Y204-MAPK3 '
  - p-T250,T255,T385,S437-MKNK1
  - p-Y FGFR2 fusion
  - 'p-Y FGFR2 fusion dimers '
  - 'p-Y194,Y195,Y272-SHC1-3 '
  - 'p-Y239,Y240,Y317-SHC1-2 '
  - 'p-Y371-CBL '
  - p-Y371-CBL:GRB2
  - 'p-Y546,Y584-PTPN11 '
  - 'p-Y55,Y227-SPRY2 '
  - p21 RAS:GDP
  - p21 RAS:GTP
  - 'phosphorylated FGFR2 L764fs*4 '
  - pre-FGFR2 mRNA:CBC
  - signaling
  - splicing complex
  - with enhanced
  license: CC0
  name: Signaling by FGFR2
seo: CreativeWork
title: Signaling by FGFR2
wpid: WP3338
---