---
annotations:
- type: Pathway Ontology
  value: DNA modification pathway
authors:
- ReactomeTeam
- Fehrhart
description: 'About 2-6% of all cytosine residues and 70-80% of cytosine residues
  in CG dinucleotides in mammalian cells are methylated at the 5 position of the pyrimidine
  ring. The cytosine residues are methylated by DNA methyltransferases after DNA replication
  and can be demethylated by passive dilution during subsequent replication or by
  active modification of the 5-methylcytosine base. Cytosine demethylation is developmentally
  regulated: one wave of demethylation occurs in primordial germ cells and one wave
  occurs by active demethylation in the male pronucleus after fertilization.<br>Some
  mechanisms of active demethylation remain controversial, however progressive oxidation
  of the methyl group of 5-methylcytosine followed by base excision by thymine DNA
  glycosylase (TDG) has been reproducibly demonstrated in vivo (reviewed in Wu and
  Zhang 2011, Franchini et al 2012, Cadet and Wagner 2013, Kohli and Zhang 2013, Ponnaluri
  et al. 2013, Rasmussen and Helin 2016). Ten-eleven translocation proteins TET1,
  TET2, and TET3 are dioxygenases that first oxidize 5-methylcytosine to 5-hydroxymethylcytosine
  (5-hmC) (Tahiliani et al. 2009, Ito et al. 2010), which is found in significant
  quantities and specific genomic locations in stem cells and neurons (Kinney and
  Pradhan 2013). TET proteins can further oxidize 5-hmC to 5-formylcytosine (5-fC)
  and then 5-carboxylcytosine (5-caC) (He et al. 2011, Ito et al. 2011). G:5-fC and
  G:5-caC base pairs are recognized by TDG, which excises the 5-fC or 5-caC and leaves
  an abasic site.<br>TET1 in mouse is expressed in neurons and its expression depends
  on neuronal activity (Guo et al. 2011, Kaas et al. 2013, Zhang et al. 2013). TET1
  is also found in embryonic stem cells (Ficz et al. 2011, Koh et al. 2011, Wu et
  al. 2011) and in primordial germ cells of mice, where it plays a role in erasure
  of imprinting (Yamaguchi et al. 2013). TET3 is expressed in oocytes and zygotes
  of mice and is required for demethylation in the male pronucleus (Gu et al. 2011,
  Iqbal et al. 2011). TET2 is the most highly expressed TET family protein in hemopoietic
  stem cells and appears to act as a tumor suppressor. TET2 is also expressed in embryonic
  stem cells (Koh et al. 2011).  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=5221030
  Reactome].'
last-edited: 2021-01-25
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP3372
- /instance/WP3372
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP3372.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: 'About 2-6% of all cytosine residues and 70-80% of cytosine residues
    in CG dinucleotides in mammalian cells are methylated at the 5 position of the
    pyrimidine ring. The cytosine residues are methylated by DNA methyltransferases
    after DNA replication and can be demethylated by passive dilution during subsequent
    replication or by active modification of the 5-methylcytosine base. Cytosine demethylation
    is developmentally regulated: one wave of demethylation occurs in primordial germ
    cells and one wave occurs by active demethylation in the male pronucleus after
    fertilization.<br>Some mechanisms of active demethylation remain controversial,
    however progressive oxidation of the methyl group of 5-methylcytosine followed
    by base excision by thymine DNA glycosylase (TDG) has been reproducibly demonstrated
    in vivo (reviewed in Wu and Zhang 2011, Franchini et al 2012, Cadet and Wagner
    2013, Kohli and Zhang 2013, Ponnaluri et al. 2013, Rasmussen and Helin 2016).
    Ten-eleven translocation proteins TET1, TET2, and TET3 are dioxygenases that first
    oxidize 5-methylcytosine to 5-hydroxymethylcytosine (5-hmC) (Tahiliani et al.
    2009, Ito et al. 2010), which is found in significant quantities and specific
    genomic locations in stem cells and neurons (Kinney and Pradhan 2013). TET proteins
    can further oxidize 5-hmC to 5-formylcytosine (5-fC) and then 5-carboxylcytosine
    (5-caC) (He et al. 2011, Ito et al. 2011). G:5-fC and G:5-caC base pairs are recognized
    by TDG, which excises the 5-fC or 5-caC and leaves an abasic site.<br>TET1 in
    mouse is expressed in neurons and its expression depends on neuronal activity
    (Guo et al. 2011, Kaas et al. 2013, Zhang et al. 2013). TET1 is also found in
    embryonic stem cells (Ficz et al. 2011, Koh et al. 2011, Wu et al. 2011) and in
    primordial germ cells of mice, where it plays a role in erasure of imprinting
    (Yamaguchi et al. 2013). TET3 is expressed in oocytes and zygotes of mice and
    is required for demethylation in the male pronucleus (Gu et al. 2011, Iqbal et
    al. 2011). TET2 is the most highly expressed TET family protein in hemopoietic
    stem cells and appears to act as a tumor suppressor. TET2 is also expressed in
    embryonic stem cells (Koh et al. 2011).  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=5221030
    Reactome].'
  keywords:
  - AscH-
  - TET1,2,3
  - 'AP-dsDNA '
  - 5-caC
  - DNA containing 5-fC
  - DNA containing 5-hmC
  - 'TDG '
  - DNA containing 5-mC
  - H2O
  - SUCCA
  - 'TET2 '
  - 5-fC
  - 'TET3 '
  - TDG
  - 'TET1 '
  - TDG:AP-dsDNA
  - CO2
  - DNA containing 5-caC
  - 'Fe2+ '
  - O2
  - 2OG
  license: CC0
  name: TET1,2,3 and TDG demethylate DNA
seo: CreativeWork
title: TET1,2,3 and TDG demethylate DNA
wpid: WP3372
---