---
annotations:
- id: PW:0000004
  parent: regulatory pathway
  type: Pathway Ontology
  value: regulatory pathway
authors:
- MaintBot
- ReactomeTeam
- Anwesha
- Mkutmon
description: 'At the center of the mammalian circadian clock is a negative transcription/translation-based
  feedback loop: The BMAL1:CLOCK/NPAS2 (ARNTL:CLOCK/NPAS2) heterodimer transactivates
  CRY and PER genes by binding E-box elements in their promoters; the CRY and PER
  proteins then inhibit transactivation by BMAL1:CLOCK/NPAS2. BMAL1:CLOCK/NPAS2 activates
  transcription of CRY, PER, and several other genes in the morning. Levels of PER
  and CRY proteins rise during the day and inhibit expression of CRY, PER, and other
  BMAL1:CLOCK/NPAS2-activated genes in the afternoon and evening. During the night
  CRY and PER proteins are targeted for degradation by phosphorylation and polyubiquitination,
  allowing the cycle to commence again in the morning. <br>Transcription of the BMAL1
  (ARNTL) gene is controlled by ROR-alpha and REV-ERBA (NR1D1), both of which are
  targets of BMAL1:CLOCK/NPAS2 in mice and both of which compete for the same element
  (RORE) in the BMAL1 promoter. ROR-alpha (RORA) activates transcription of BMAL1;
  REV-ERBA represses transcription of BMAL1. This mutual control forms a secondary,
  reinforcing loop of the circadian clock. REV-ERBA shows strong circadian rhythmicity
  and confers circadian expression on BMAL1. <br>BMAL1 can form heterodimers with
  either CLOCK or NPAS2, which act redundantly but show different tissue specificity.
  The BMAL1:CLOCK and BMAL1:NPAS2 heterodimers activate a set of genes that possess
  E-box elements (consensus CACGTG) in their promoters. This confers circadian expression
  on the genes. The PER genes (PER1, PER2, PER3) and CRY genes (CRY1, CRY2) are among
  those activated by BMAL1:CLOCK and BMAL1:NPAS2. PER and CRY mRNA accumulates during
  the morning and the proteins accumulate during the afternoon. PER and CRY proteins
  form complexes in the cytosol and these are bound by either CSNK1D or CSNK1E kinases
  which phosphorylate PER and CRY. The phosphorylated PER:CRY:kinase complex is translocated
  into the nucleus due to the nuclear localization signal of PER and CRY. Within the
  nucleus the PER:CRY complexes bind BMAL1:CLOCK and BMAL1:NPAS2, inhibiting their
  transactivation activity and their phosphorylation. This reduces expression of the
  target genes of BMAL1:CLOCK and BMAL1:NPAS2 during the afternoon and evening. <br>PER:CRY
  complexes also traffic out of the nucleus into the cytosol due to the nuclear export
  signal of PER. During the night PER:CRY complexes are polyubiquitinated and degraded,
  allowing the cycle to begin again. Phosphorylated PER is bound by Beta-TrCP1, a
  cytosolic F-box type component of some SCF E3 ubiquitin ligases. CRY is bound by
  FBXL3, a nucleoplasmic F-box type component of some SCF E3 ubiquitin ligases. Phosphorylation
  of CRY1 by Adenosine monophosphate-activated kinase (AMPK) enhances degradation
  of CRY1. PER and CRY are subsequently polyubiquitinated and proteolyzed by the 26S
  proteasome.<br>The circadian clock is cell-autonomous and some, but not all cells
  of the body exhibit circadian rhythms in metabolism, cell division, and gene transcription.
  The suprachiasmatic nucleus (SCN) in the hypothalamus is the major clock in the
  body and receives its major input from light (via retinal neurons) and a minor input
  from nutrient intake.  The SCN and other brain tissues determine waking and feeding
  cycles and influence the clocks in other tissues by hormone secretion and nervous
  stimulation. Independently of the SCN, other tissues such as liver receive inputs
  from signals from the brain and from nutrients.  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=400253
  Reactome].'
last-edited: 2021-01-25
organisms:
- Homo sapiens
redirect_from:
- /index.php/Pathway:WP1797
- /instance/WP1797
revision: null
schema-jsonld:
- '@context': https://schema.org/
  '@id': https://wikipathways.github.io/pathways/WP1797.html
  '@type': Dataset
  creator:
    '@type': Organization
    name: WikiPathways
  description: 'At the center of the mammalian circadian clock is a negative transcription/translation-based
    feedback loop: The BMAL1:CLOCK/NPAS2 (ARNTL:CLOCK/NPAS2) heterodimer transactivates
    CRY and PER genes by binding E-box elements in their promoters; the CRY and PER
    proteins then inhibit transactivation by BMAL1:CLOCK/NPAS2. BMAL1:CLOCK/NPAS2
    activates transcription of CRY, PER, and several other genes in the morning. Levels
    of PER and CRY proteins rise during the day and inhibit expression of CRY, PER,
    and other BMAL1:CLOCK/NPAS2-activated genes in the afternoon and evening. During
    the night CRY and PER proteins are targeted for degradation by phosphorylation
    and polyubiquitination, allowing the cycle to commence again in the morning. <br>Transcription
    of the BMAL1 (ARNTL) gene is controlled by ROR-alpha and REV-ERBA (NR1D1), both
    of which are targets of BMAL1:CLOCK/NPAS2 in mice and both of which compete for
    the same element (RORE) in the BMAL1 promoter. ROR-alpha (RORA) activates transcription
    of BMAL1; REV-ERBA represses transcription of BMAL1. This mutual control forms
    a secondary, reinforcing loop of the circadian clock. REV-ERBA shows strong circadian
    rhythmicity and confers circadian expression on BMAL1. <br>BMAL1 can form heterodimers
    with either CLOCK or NPAS2, which act redundantly but show different tissue specificity.
    The BMAL1:CLOCK and BMAL1:NPAS2 heterodimers activate a set of genes that possess
    E-box elements (consensus CACGTG) in their promoters. This confers circadian expression
    on the genes. The PER genes (PER1, PER2, PER3) and CRY genes (CRY1, CRY2) are
    among those activated by BMAL1:CLOCK and BMAL1:NPAS2. PER and CRY mRNA accumulates
    during the morning and the proteins accumulate during the afternoon. PER and CRY
    proteins form complexes in the cytosol and these are bound by either CSNK1D or
    CSNK1E kinases which phosphorylate PER and CRY. The phosphorylated PER:CRY:kinase
    complex is translocated into the nucleus due to the nuclear localization signal
    of PER and CRY. Within the nucleus the PER:CRY complexes bind BMAL1:CLOCK and
    BMAL1:NPAS2, inhibiting their transactivation activity and their phosphorylation.
    This reduces expression of the target genes of BMAL1:CLOCK and BMAL1:NPAS2 during
    the afternoon and evening. <br>PER:CRY complexes also traffic out of the nucleus
    into the cytosol due to the nuclear export signal of PER. During the night PER:CRY
    complexes are polyubiquitinated and degraded, allowing the cycle to begin again.
    Phosphorylated PER is bound by Beta-TrCP1, a cytosolic F-box type component of
    some SCF E3 ubiquitin ligases. CRY is bound by FBXL3, a nucleoplasmic F-box type
    component of some SCF E3 ubiquitin ligases. Phosphorylation of CRY1 by Adenosine
    monophosphate-activated kinase (AMPK) enhances degradation of CRY1. PER and CRY
    are subsequently polyubiquitinated and proteolyzed by the 26S proteasome.<br>The
    circadian clock is cell-autonomous and some, but not all cells of the body exhibit
    circadian rhythms in metabolism, cell division, and gene transcription. The suprachiasmatic
    nucleus (SCN) in the hypothalamus is the major clock in the body and receives
    its major input from light (via retinal neurons) and a minor input from nutrient
    intake.  The SCN and other brain tissues determine waking and feeding cycles and
    influence the clocks in other tissues by hormone secretion and nervous stimulation.
    Independently of the SCN, other tissues such as liver receive inputs from signals
    from the brain and from nutrients.  View original pathway at [http://www.reactome.org/PathwayBrowser/#DIAGRAM=400253
    Reactome].'
  keywords:
  - '11DCORST '
  - 'AA '
  - ADP
  - 'ALA '
  - 'ALDO '
  - ARNTL
  - 'ARNTL '
  - ARNTL gene
  - 'ARNTL gene '
  - ATP
  - 'BHLHE41 '
  - BMAL1:CLOCK,NPAS2
  - BMAL1:CLOCK,NPAS2:CRY
  - BMAL1:CLOCK,NPAS2:CRY:PER
  - BTRC
  - 'BTRC '
  - BTRC:CUL1:SKP1
  - Beta-TrCP1:PER
  - 'CARM1 '
  - 'CHD9 '
  - CLOCK
  - 'CLOCK '
  - CLOCK gene
  - 'CLOCK gene '
  - CLOCK,NPAS2
  - 'CORST '
  - 'CORT '
  - 'CREBBP '
  - 'CREM isoform 6 (ICER1) '
  - CREM:PER1 gene
  - 'CRTC1 '
  - 'CRTC2 '
  - 'CRTC3 '
  - CRY1
  - 'CRY1 '
  - CRY1 gene
  - 'CRY1 gene '
  - CRY1,2
  - CRY2
  - 'CRY2 '
  - CRY2 gene
  - CRY:PER:Kinase
  - 'CSNK1D '
  - 'CSNK1E '
  - CSNK1E,CSNK1D
  - 'CUL1 '
  - Coactivator complex
  - EP300
  - 'EP300 '
  - 'EPA '
  - FBXL3
  - 'FBXL3 '
  - FBXL3:CRY
  - HDAC3
  - 'HDAC3 '
  - 'HELZ2 '
  - HIF1A
  - 'LINA '
  - 'MED1 '
  - 'MEF2C '
  - MEF2C,D:PPARGC1A
  - 'MEF2D '
  - 'NCOA1 '
  - 'NCOA2 '
  - 'NCOA6 '
  - NCOR1
  - 'NCOR1 '
  - 'NFIL3 '
  - NFIL3:BHLHE41:PER2
  - NPAS2
  - 'NPAS2 '
  - NPAS2 gene
  - 'NPAS2 gene '
  - NR1D1
  - 'NR1D1 '
  - NR1D1 (REV-ERBA)
  - NR1D1 gene
  - 'NR1D1 gene '
  - NR1D1:heme:Corepressors:ARNTL gene
  - NR1D1:heme:Corepressors:CLOCK gene
  - NR1D1:heme:Corepressors:NPAS2 gene
  - NR1D1:heme:Corepressors:NR1D1 gene
  - NR1D1:heme:Corepressors:PPARGC1A gene
  - 'NR3C1 '
  - NR3C1:(ALDO,11DCORST,CORST,CORT) dimer
  - NRIP1
  - 'NRIP1 '
  - PER1
  - 'PER1 '
  - PER1 gene
  - 'PER1 gene '
  - 'PER1 mRNA '
  - PER1,PER2
  - PER2
  - 'PER2 '
  - PER2 gene
  - 'PER2 gene '
  - PP1 catalytic
  - 'PPARA '
  - PPARA:RXRA
  - PPARGC1A
  - 'PPARGC1A '
  - PPARGC1A gene
  - 'PPARGC1A gene '
  - 'PPP1CA '
  - 'PPP1CB '
  - 'PPP1CC '
  - 'Palm '
  - 'Peroxisome Proliferator Receptor Element (PPRE) '
  - RAI1
  - 'RBM4 '
  - RBM4:PER1 mRNA
  - RORA
  - 'RORA '
  - RORA activates gene
  - RORA gene
  - 'RORA gene '
  - RORA:Coactivator:ARNTL gene
  - RORA:Coactivator:NR1D1 gene
  - RORA:EP300:NPAS2
  - 'RPS27A(1-76) '
  - 'RXRA '
  - SIK1
  - SIRT1
  - 'SKP1 '
  - 'SMARCD3 '
  - 'TBL1X '
  - 'TBL1XR1 '
  - 'TGS1 '
  - 'UBA52(1-76) '
  - 'UBB(1-76) '
  - 'UBB(153-228) '
  - 'UBB(77-152) '
  - 'UBC(1-76) '
  - 'UBC(153-228) '
  - 'UBC(229-304) '
  - 'UBC(305-380) '
  - 'UBC(381-456) '
  - 'UBC(457-532) '
  - 'UBC(533-608) '
  - 'UBC(609-684) '
  - 'UBC(77-152) '
  - Ub
  - 'Ub-p-S-CRY1 '
  - 'Ub-p-S-CRY2 '
  - 'Ub-p-S-PER1 '
  - 'Ub-p-S-PER2 '
  - activates circadian
  - expression
  - ferriheme b
  - 'ferriheme b '
  - gene
  - gene expression
  - p-BMAL1:p-CLOCK,NPAS2
  - p-BMAL1:p-CLOCK,NPAS2:CRY1 gene
  - p-BMAL1:p-CLOCK,NPAS2:NR1D1 gene
  - p-BMAL1:p-CLOCK,NPAS2:PER1 gene
  - p-BMAL1:p-CLOCK,NPAS2:PER2 gene
  - p-BMAL1:p-CLOCK,NPAS2:RORA gene
  - p-CREB:CRTC1:PER1
  - p-CRY1,p-CRY2
  - p-CRY:p-PER:Kinase
  - p-PER1,p-PER2
  - p-PPARGC1A
  - 'p-S-ARNTL '
  - 'p-S-CLOCK '
  - 'p-S-CRY1 '
  - 'p-S-CRY2 '
  - 'p-S-NPAS2 '
  - 'p-S-PER1 '
  - 'p-S-PER2 '
  - 'p-S133-CREB1 '
  - p-S133-CREB:CRTC1,2,3:PPARGC1A gene
  - 'p-T178,S539-PPARGC1A '
  - 'p-T263,S266,T299-PPARGC1A '
  - p-T69,T71-ATF2
  - represses gene
  - subunit
  - ub-p-CRY1,ub-p-CRY2
  - ub-p-PER1,ub-p-PER2
  license: CC0
  name: Circadian Clock
seo: CreativeWork
title: Circadian Clock
wpid: WP1797
---